msgid ""
msgstr ""
"Project-Id-Version: Fedora Localization Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-03-27 15:38-0400\n"
"PO-Revision-Date: \n"
"Last-Translator: Victor Ibragimov <victor.ibragimov@gmail.com>\n"
"Language-Team: Tajik Fedora & Software Localization Team <victor."
"ibragimov@gmail.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-Language: Tajik\n"
"X-Poedit-Country: TAJIKISTAN\n"
"X-Poedit-SourceCharset: utf-8\n"

#: ../anaconda:266 ../anaconda:312
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr ""

#: ../anaconda:269 ../anaconda:315
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr ""

#: ../anaconda:283 ../anaconda:327 ../cmdline.py:86 ../gui.py:1251
#: ../text.py:543
#, python-format
msgid ""
"The following error was found while parsing your kickstart configuration:\n"
"\n"
"%s"
msgstr ""

#: ../anaconda:449
#, fuzzy
msgid "Press <enter> for a shell"
msgstr "Номро барои нусхаи аслӣ ворид кунед"

#: ../anaconda:458
#, fuzzy
msgid "Fatal Error"
msgstr "Хато"

#: ../anaconda:459
#, python-format
msgid ""
"You do not have enough RAM to install %s on this machine.\n"
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: ../anaconda:464 ../rescue.py:285 ../rescue.py:311 ../rescue.py:324
#: ../rescue.py:397 ../rescue.py:406 ../text.py:694
#: ../textw/constants_text.py:44 ../loader/cdinstall.c:228
#: ../loader/cdinstall.c:232 ../loader/cdinstall.c:248
#: ../loader/cdinstall.c:251 ../loader/cdinstall.c:445
#: ../loader/cdinstall.c:452 ../loader/cdinstall.c:517
#: ../loader/dirbrowser.c:145 ../loader/driverdisk.c:98
#: ../loader/driverdisk.c:243 ../loader/driverdisk.c:274
#: ../loader/driverdisk.c:305 ../loader/driverdisk.c:340
#: ../loader/driverdisk.c:359 ../loader/driverdisk.c:371
#: ../loader/driverdisk.c:379 ../loader/driverdisk.c:539
#: ../loader/driverdisk.c:576 ../loader/driverselect.c:81
#: ../loader/driverselect.c:205 ../loader/hdinstall.c:143
#: ../loader/hdinstall.c:289 ../loader/hdinstall.c:352
#: ../loader/hdinstall.c:384 ../loader/hdinstall.c:454
#: ../loader/hdinstall.c:497 ../loader/hdinstall.c:510 ../loader/kbd.c:138
#: ../loader/kickstart.c:132 ../loader/kickstart.c:142
#: ../loader/kickstart.c:185 ../loader/kickstart.c:284
#: ../loader/kickstart.c:375 ../loader/kickstart.c:553 ../loader/lang.c:115
#: ../loader/lang.c:378 ../loader/loader.c:402 ../loader/loader.c:438
#: ../loader/loader.c:481 ../loader/loader.c:502 ../loader/loader.c:519
#: ../loader/loader.c:556 ../loader/loader.c:1203 ../loader/loader.c:1378
#: ../loader/mediacheck.c:46 ../loader/mediacheck.c:85
#: ../loader/mediacheck.c:92 ../loader/mediacheck.c:101 ../loader/method.c:279
#: ../loader/method.c:353 ../loader/modules.c:335 ../loader/modules.c:347
#: ../loader/net.c:564 ../loader/net.c:937 ../loader/net.c:1541
#: ../loader/net.c:1564 ../loader/net.c:1808 ../loader/net.c:1830
#: ../loader/net.c:1842 ../loader/nfsinstall.c:91 ../loader/nfsinstall.c:248
#: ../loader/nfsinstall.c:267 ../loader/nfsinstall.c:335
#: ../loader/telnetd.c:92 ../loader/urlinstall.c:83 ../loader/urlinstall.c:107
#: ../loader/urlinstall.c:175 ../loader/urlinstall.c:418
#: ../loader/urlinstall.c:427 ../loader/urlinstall.c:436 ../loader/urls.c:292
#: ../loader/urls.c:328 ../loader/urls.c:334 ../loader/urls.c:346
msgid "OK"
msgstr "OK"

#: ../anaconda:471
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""

#: ../anaconda:534
msgid "Starting graphical installation..."
msgstr "Оғози сабткунии графикӣ..."

#: ../anaconda:823
msgid "Would you like to use VNC?"
msgstr "Оё VNC*ро истифода мебаред?"

#: ../anaconda:824
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""

#: ../anaconda:852
msgid "Install class forcing text mode installation"
msgstr ""

#: ../anaconda:871
msgid "Graphical installation not available...  Starting text mode."
msgstr "Сабткунии графикӣ дастнорас аст... Оғози ҳолати матнӣ."

#: ../anaconda:879
msgid "DISPLAY variable not set. Starting text mode!"
msgstr ""

#: ../anaconda:1047
msgid "reipl configuration successful => reboot"
msgstr ""

#: ../anaconda:1050
msgid "reipl configuration failed => halt"
msgstr ""

#: ../backend.py:144
#, python-format
msgid "Upgrading %s\n"
msgstr "Навсозии %s\n"

#: ../backend.py:146
#, python-format
msgid "Installing %s\n"
msgstr "Сабткунии %s\n"

#: ../backend.py:166
msgid "Copying File"
msgstr "Нусхабардории файл"

#: ../backend.py:167
msgid "Transferring install image to hard drive..."
msgstr ""

#: ../backend.py:177
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is probably due to bad media."
msgstr ""

#: ../backend.py:181
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""

#: ../backend.py:185 ../image.py:285 ../livecd.py:189 ../livecd.py:382
#: ../partIntfHelpers.py:202 ../text.py:460 ../text.py:464
#: ../yuminstall.py:371 ../yuminstall.py:664 ../yuminstall.py:779
#: ../yuminstall.py:1056 ../yuminstall.py:1124 ../yuminstall.py:1312
#: ../yuminstall.py:1337 ../iw/autopart_type.py:100 ../iw/autopart_type.py:199
#: ../iw/autopart_type.py:328 ../iw/autopart_type.py:331
#: ../iw/autopart_type.py:360 ../iw/osbootwidget.py:209
#: ../iw/osbootwidget.py:218 ../iw/raid_dialog_gui.py:700
#: ../iw/raid_dialog_gui.py:739 ../iw/task_gui.py:63 ../iw/task_gui.py:161
#: ../iw/task_gui.py:292 ../iw/task_gui.py:414 ../textw/netconfig_text.py:271
#: ../textw/partition_text.py:178 ../textw/partition_text.py:184
#: ../textw/partition_text.py:206 ../textw/upgrade_text.py:175
#: ../textw/upgrade_text.py:182 ../storage/__init__.py:268
#: ../storage/__init__.py:1423 ../storage/__init__.py:1439
#: ../loader/cdinstall.c:228 ../loader/cdinstall.c:517
#: ../loader/driverdisk.c:98 ../loader/driverdisk.c:305
#: ../loader/driverdisk.c:340 ../loader/driverdisk.c:371
#: ../loader/driverdisk.c:379 ../loader/driverdisk.c:443
#: ../loader/hdinstall.c:143 ../loader/hdinstall.c:352
#: ../loader/hdinstall.c:454 ../loader/hdinstall.c:497
#: ../loader/hdinstall.c:510 ../loader/kickstart.c:284 ../loader/lang.c:115
#: ../loader/loader.c:402 ../loader/loader.c:519 ../loader/loader.c:1203
#: ../loader/mediacheck.c:46 ../loader/mediacheck.c:85
#: ../loader/mediacheck.c:92 ../loader/method.c:279 ../loader/method.c:353
#: ../loader/nfsinstall.c:248 ../loader/nfsinstall.c:267
#: ../loader/telnetd.c:92 ../loader/urlinstall.c:83 ../loader/urlinstall.c:107
#: ../loader/urlinstall.c:175 ../loader/urls.c:328 ../loader/urls.c:334
msgid "Error"
msgstr "Хато"

#: ../bootloader.py:43 ../bootloader.py:198 ../image.py:82
#: ../partedUtils.py:169 ../partedUtils.py:199 ../partedUtils.py:835
#: ../partedUtils.py:985 ../upgrade.py:298 ../yuminstall.py:1510
#: ../yuminstall.py:1539 ../iw/autopart_type.py:209 ../iw/blpasswidget.py:151
#: ../iw/upgrade_swap_gui.py:192 ../iw/upgrade_swap_gui.py:200
#: ../iw/upgrade_swap_gui.py:207 ../textw/upgrade_text.py:187
#: ../storage/devicetree.py:125 ../storage/devicetree.py:150
msgid "Warning"
msgstr "Огоҳӣ"

#: ../bootloader.py:44
msgid ""
"Your filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""

#: ../bootloader.py:48 ../exception.py:468 ../exception.py:485
#: ../exception.py:508 ../exception.py:523 ../exception.py:548 ../gui.py:1256
#: ../gui.py:1384 ../gui.py:1461 ../image.py:91 ../livecd.py:196
#: ../livecd.py:389 ../packages.py:127 ../packages.py:135 ../packages.py:143
#: ../packages.py:151 ../packages.py:163 ../packages.py:172 ../packages.py:180
#: ../packages.py:189 ../upgrade.py:100 ../upgrade.py:120 ../upgrade.py:253
#: ../yuminstall.py:783 ../yuminstall.py:863 ../yuminstall.py:869
#: ../yuminstall.py:1027 ../yuminstall.py:1052 ../yuminstall.py:1112
#: ../yuminstall.py:1307 ../yuminstall.py:1344 ../storage/__init__.py:85
#: ../storage/__init__.py:1508
msgid "_Exit installer"
msgstr "_Хуруҷи сабткунанда"

#: ../bootloader.py:48 ../image.py:92 ../image.py:248 ../kickstart.py:1214
#: ../kickstart.py:1253 ../partedUtils.py:988 ../upgrade.py:100
#: ../upgrade.py:120 ../yuminstall.py:1117 ../iw/partition_gui.py:1036
#: ../storage/__init__.py:1509
msgid "_Continue"
msgstr "_Давом диҳед"

#: ../bootloader.py:105 ../platform.py:90 ../platform.py:253
#: ../platform.py:297 ../platform.py:358
msgid "RAID Device"
msgstr "Дастгоҳи RAID"

#: ../bootloader.py:134
msgid "Bootloader"
msgstr ""

#: ../bootloader.py:134
#, fuzzy
msgid "Installing bootloader..."
msgstr "Чопгари %1 танзим шуда истодааст"

#: ../bootloader.py:199
msgid ""
"No kernel packages were installed on your system.  Your boot loader "
"configuration will not be changed."
msgstr ""

#: ../cmdline.py:55
msgid "Completed"
msgstr "Тайёр шуд"

#: ../cmdline.py:63
#, fuzzy
msgid "In progress...   "
msgstr "Кор рафта истодааст..."

#: ../cmdline.py:98
msgid "Can't have a question in command line mode!"
msgstr ""

#: ../constants.py:74
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""

#: ../constants.py:80
#, fuzzy
msgid " with the provider of this software."
msgstr "Бурида гузаштан бо ин хати каҷи конусшакл"

#: ../constants.py:84
#, fuzzy, python-format
msgid " against anaconda at %s"
msgstr "Истодан дар чуқури"

#: ../exception.py:391
#, python-format
msgid ""
"Your bug could not be filed due to the following error when communicating "
"with bugzilla:\n"
"\n"
"%s"
msgstr ""

#: ../exception.py:394
#, python-format
msgid ""
"Your bug could not be filed due to bad information in the bug fields.  This "
"is most likely an error in anaconda:\n"
"\n"
"%s"
msgstr ""

#: ../exception.py:398
msgid "Unable To File Bug"
msgstr "Гузариши хато эҷод карда намешавад"

#: ../exception.py:404
msgid "Bug Filing Not Supported"
msgstr "Гузариши хато пуштибонӣ намешавад"

#: ../exception.py:405
msgid ""
"Your distribution does not provide a supported bug filing system, so you "
"cannot save your exception this way."
msgstr ""

#: ../exception.py:411
msgid "Invalid Bug Information"
msgstr "Иттилооти хатои нодуруст"

#: ../exception.py:412
msgid "Please provide a valid username, password, and short bug description."
msgstr ""

#: ../exception.py:424
msgid "Unable To Login"
msgstr "Ворид карда намешавад"

#: ../exception.py:425
#, python-format
msgid ""
"There was an error logging into %s using the provided username and password."
msgstr ""

#: ../exception.py:462
#, fuzzy
msgid "Bug Created"
msgstr "<i>Эҷод кардашудагӣ дар</i>: %1"

#: ../exception.py:463
#, python-format
msgid ""
"A new bug has been created with your traceback attached. Please add "
"additional information such as what you were doing when you encountered the "
"bug, screenshots, and whatever else is appropriate to the following bug:\n"
"\n"
"%s/%s"
msgstr ""

#: ../exception.py:479
msgid "Bug Updated"
msgstr "Хато ислоҳ карда шуд"

#: ../exception.py:480
#, python-format
msgid ""
"A bug with your information already exists.  Your account has been added to "
"the CC list and your traceback added as a comment.  Please add additional "
"descriptive information to the following bug:\n"
"\n"
"%s/%s"
msgstr ""

#: ../exception.py:504 ../exception.py:519 ../exception.py:544
#, fuzzy
msgid "Dump Written"
msgstr "<p><i>Навистаги %1.</i></p>"

#: ../exception.py:505 ../exception.py:520
msgid ""
"Your system's state has been successfully written to the disk. The installer "
"will now exit."
msgstr ""

#: ../exception.py:511 ../exception.py:528 ../exception.py:551
#, fuzzy
msgid "Dump Not Written"
msgstr ""
"%1 навишта буд бо\n"
"%2"

#: ../exception.py:512 ../exception.py:529
msgid "There was a problem writing the system state to the disk."
msgstr ""

#: ../exception.py:535 ../exception.py:558 ../rescue.py:201
#: ../yuminstall.py:1022
msgid "No Network Available"
msgstr "Интернет дастнорас аст"

#: ../exception.py:536 ../exception.py:559
msgid ""
"Cannot save a bug report since there is no active networking device "
"available."
msgstr ""

#: ../exception.py:545
msgid ""
"Your system's state has been successfully written to the remote host.  The "
"installer will now exit."
msgstr ""

#: ../exception.py:552
msgid "There was a problem writing the system state to the remote host."
msgstr ""

#: ../gui.py:106
msgid "An error occurred copying the screenshots over."
msgstr ""

#: ../gui.py:117
#, fuzzy
msgid "Screenshots Copied"
msgstr "Нав карда шуд, нусха карда шуд"

#: ../gui.py:118
msgid ""
"The screenshots have been saved into the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""

#: ../gui.py:161
#, fuzzy
msgid "Saving Screenshot"
msgstr "Нигоҳ куни нашуд."

#: ../gui.py:162
#, fuzzy, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "Бозии шумо захира карда шудааст."

#: ../gui.py:165
#, fuzzy
msgid "Error Saving Screenshot"
msgstr "Хатогӣ Ҳангоми Захиракунӣ"

#: ../gui.py:166
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""

#: ../gui.py:546 ../text.py:500
msgid "Installation Key"
msgstr "Калиди сабткунӣ"

#: ../gui.py:617 ../text.py:308
msgid ""
"Choose a passphrase for your encrypted devices. You will be prompted for the "
"passphrase during system boot."
msgstr ""

#: ../gui.py:636 ../gui.py:644 ../text.py:346 ../text.py:356
#, fuzzy
msgid "Error with passphrase"
msgstr "Su вернул ошибку.\n"

#: ../gui.py:637 ../text.py:347
#, fuzzy
msgid "The passphrases you entered were different.  Please try again."
msgstr "Паролҳо баробар наомаданд. Лутфан, боз кӯшиш кунед."

#: ../gui.py:645
msgid "The passphrase must be at least eight characters long."
msgstr ""

#: ../gui.py:679 ../text.py:380
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: ../gui.py:736
#, fuzzy, python-format
msgid "Bugzilla (%s)"
msgstr "Тафсири Bugzilla:"

#: ../gui.py:766
#, fuzzy
msgid "No devices found"
msgstr "Устройства PCI не найдены."

#: ../gui.py:873 ../gui.py:1461 ../partIntfHelpers.py:149
#: ../partIntfHelpers.py:304 ../text.py:98 ../text.py:99
#: ../textw/constants_text.py:48 ../loader/dirbrowser.c:145
#: ../loader/driverdisk.c:244 ../loader/kickstart.c:375 ../loader/loader.c:438
#: ../loader/loader.c:556
msgid "Cancel"
msgstr "Бекор кардан"

#: ../gui.py:885 ../gui.py:886 ../gui.py:998 ../gui.py:999
#: tmp/anaconda.glade.h:2
#, fuzzy
msgid "_Debug"
msgstr "Ғалатёб"

#: ../gui.py:1049 ../gui.py:1051 ../gui.py:1458 ../livecd.py:113
msgid "Exit installer"
msgstr "Хуруҷи сабткунанда"

#: ../gui.py:1051 ../text.py:283 ../text.py:291
#, fuzzy
msgid "Debug"
msgstr "Ғалатёб"

#: ../gui.py:1053 ../text.py:287
#, fuzzy
msgid "Exception Occurred"
msgstr ""
"Хатогӣ рӯй дод:\n"
"%1"

#: ../gui.py:1253 ../text.py:545
#, fuzzy
msgid "Error Parsing Kickstart Config"
msgstr "kppp: хатогӣ дар гузориши қоидаҳо\n"

#: ../gui.py:1293
#, fuzzy
msgid "default:LTR"
msgstr "Сатри тасвиркунии нобаён"

#: ../gui.py:1373 ../text.py:659
msgid "Error!"
msgstr "Хато!"

#: ../gui.py:1374 ../text.py:660
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""

#: ../gui.py:1379 ../image.py:164 ../image.py:194 ../packages.py:428
#: ../packages.py:433
msgid "_Exit"
msgstr "_Хуруҷ"

#: ../gui.py:1380 ../image.py:164 ../image.py:194 ../livecd.py:196
#: ../yuminstall.py:656 ../yuminstall.py:1052 ../yuminstall.py:1112
#: ../yuminstall.py:1307
msgid "_Retry"
msgstr "_Кӯшиши нав"

#: ../gui.py:1383 ../packages.py:432
#, fuzzy
msgid "The installer will now exit..."
msgstr ""
"\n"
"Хурӯҷшавӣ аз KWin..."

#: ../gui.py:1386 ../packages.py:435
msgid "Your system will now be rebooted..."
msgstr ""

#: ../gui.py:1387 ../image.py:248 ../packages.py:436 ../partedUtils.py:987
#: ../yuminstall.py:1360
#, fuzzy
msgid "_Reboot"
msgstr "перезагрузить"

#: ../gui.py:1389
msgid "Exiting"
msgstr "Хуруҷ мекунад"

#: ../gui.py:1459
#, fuzzy
msgid "Are you sure you wish to exit the installation?"
msgstr "Дар ҳақиқат мехоҳед, ки навиштаҷоти '%1'-ро нобуд созед?"

#: ../gui.py:1468
#, python-format
msgid "%s Installer"
msgstr "Сабткунандаи %s"

#: ../gui.py:1475
#, fuzzy
msgid "Unable to load title bar"
msgstr "Файли аудио бор карда намешавад"

#: ../gui.py:1531
msgid "Install Window"
msgstr "Тирезаи сабт"

#: ../image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: ../image.py:156
msgid "Couldn't Mount ISO Source"
msgstr ""

#: ../image.py:157
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""

#: ../image.py:185
msgid "Missing ISO 9660 Image"
msgstr ""

#: ../image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""

#: ../image.py:241
#, fuzzy
msgid "Required Install Media"
msgstr "Озмоиши медиа..."

#: ../image.py:242
#, python-format
msgid ""
"The software you have selected to install will require the following %s %s "
"discs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""

#: ../image.py:248 ../livecd.py:388 ../packages.py:382 ../packages.py:433
#: ../packages.py:436 ../upgrade.py:252 ../yuminstall.py:783
#: ../yuminstall.py:869 ../yuminstall.py:1344 ../yuminstall.py:1360
msgid "_Back"
msgstr "_Ақиб"

#: ../image.py:286
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: ../installclass.py:71
#, fuzzy
msgid "Install on System"
msgstr "Намуди системаи файлҳо:"

#: ../iutil.py:740
#, python-format
msgid "Error: On open, cannot set reIPL method to %s (%s: %s)"
msgstr ""

#: ../iutil.py:748
#, python-format
msgid "Error: On write, cannot set reIPL method to %s (%s: %s)"
msgstr ""

#: ../iutil.py:755
#, python-format
msgid "Error: On close, cannot set reIPL method to %s (%s: %s)"
msgstr ""

#: ../iutil.py:765 ../iutil.py:817
#, python-format
msgid "Error: %s splits into %s but not like we expect"
msgstr ""

#: ../iutil.py:778
#, python-format
msgid "Error: Could not set %s as reIPL device (%s)"
msgstr ""

#: ../iutil.py:787
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr ""

#: ../iutil.py:796
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr ""

#: ../iutil.py:806
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""

#: ../iutil.py:834
#, python-format
msgid "Error: reading FCP property %s for reIPL (%s)"
msgstr ""

#: ../iutil.py:847
#, python-format
msgid "Error: writing FCP property %s for reIPL (%s)"
msgstr ""

#: ../iutil.py:860
#, python-format
msgid "Error: writing default FCP property %s for reIPL (%s)"
msgstr ""

#: ../iutil.py:870
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN %"
"(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""

#: ../iutil.py:877
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""

#: ../iutil.py:887
msgid "Could not get information for mount point /boot or /"
msgstr ""

#: ../iutil.py:897
msgid "Error determining mount point type"
msgstr ""

#: ../iutil.py:901
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr ""

#: ../kickstart.py:96
#, python-format
msgid ""
"There was an error running the kickstart script at line %s.  You may examine "
"the output in %s.  This is a fatal error and your install will be aborted.  "
"Press the OK button to exit the installer."
msgstr ""

#: ../kickstart.py:111 ../kickstart.py:113
#, fuzzy
msgid "Scriptlet Failure"
msgstr "Хатои Kprocess"

#: ../kickstart.py:1155 ../kickstart.py:1172
msgid "Running..."
msgstr "Иҷро мекунад..."

#: ../kickstart.py:1156
#, fuzzy
msgid "Running post-install scripts"
msgstr "Utility for running KJSEmbed scripts \n"

#: ../kickstart.py:1173
#, fuzzy
msgid "Running pre-install scripts"
msgstr "Ба кор даровардани пеш-сардиҳии фармон..."

#: ../kickstart.py:1205
#, fuzzy
msgid "Missing Package"
msgstr "Сабткунии қуттиҳо"

#: ../kickstart.py:1206
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort your installation?"
msgstr ""

#: ../kickstart.py:1212 ../kickstart.py:1251
msgid "_Abort"
msgstr "_Қатъ"

#: ../kickstart.py:1213 ../kickstart.py:1252
#, fuzzy
msgid "_Ignore All"
msgstr "Рад кардан"

#: ../kickstart.py:1243
msgid "Missing Group"
msgstr "Гуруҳ мавҷуд нест"

#: ../kickstart.py:1244
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort your installation?"
msgstr ""

#: ../kickstart.py:1350
#, python-format
msgid ""
"Your kickstart file is missing required information that anaconda cannot "
"prompt for.  Please add the following sections and try again:\n"
"%s"
msgstr ""

#: ../livecd.py:108
#, fuzzy
msgid "Unable to find image"
msgstr "Тасвирро чоп карда натавонист."

#: ../livecd.py:109
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: ../livecd.py:171
msgid "Copying live image to hard drive."
msgstr ""

#: ../livecd.py:190
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""

#: ../livecd.py:220
#, fuzzy
msgid "Doing post-installation"
msgstr "Оғози сабткунии графикӣ..."

#: ../livecd.py:221
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes..."
msgstr ""

#: ../livecd.py:383
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""

#: ../network.py:58
msgid "Hostname must be 255 or fewer characters in length."
msgstr ""

#: ../network.py:64
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-"
"9'"
msgstr ""

#: ../network.py:69
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""

#: ../network.py:178
#, fuzzy
msgid "IP address is missing."
msgstr "Суроғаи фиристанда мавҷуд нест."

#: ../network.py:182
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""

#: ../network.py:185
#, fuzzy, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "Суроғаи ворид кардаи шумо дуруст нест."

#: ../network.py:187
#, fuzzy, python-format
msgid "'%s' is an invalid IP address."
msgstr "Номи соҳиб/IP-Суроға нест"

#: ../packages.py:122
#, fuzzy
msgid "Device Resize Failed"
msgstr "Васлкунӣ қатъ карда шуд"

#: ../packages.py:123
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr ""

#: ../packages.py:130
#, fuzzy
msgid "Device Creation Failed"
msgstr "Файли муайянкунӣ"

#: ../packages.py:131
#, python-format
msgid "An error was encountered while creating device %s."
msgstr ""

#: ../packages.py:138
msgid "Device Removal Failed"
msgstr ""

#: ../packages.py:139
#, python-format
msgid "An error was encountered while removing device %s."
msgstr ""

#: ../packages.py:146
msgid "Device Setup Failed"
msgstr ""

#: ../packages.py:147
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr ""

#: ../packages.py:158
#, fuzzy
msgid "Resizing Failed"
msgstr "Васлкунӣ қатъ карда шуд"

#: ../packages.py:159
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr ""

#: ../packages.py:166
#, fuzzy
msgid "Migration Failed"
msgstr "<h3>Амалиётро ба кор бурда нашуд!</h3><p>%1</p>"

#: ../packages.py:167
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr ""

#: ../packages.py:175
#, fuzzy
msgid "Formatting Failed"
msgstr "Шаклбандӣ"

#: ../packages.py:176
#, fuzzy, python-format
msgid "An error was encountered while formatting device %s."
msgstr "Интизор шавед то даме, ки ба хидматрасон дархост меравад..."

#: ../packages.py:184
msgid "Storage Activation Failed"
msgstr ""

#: ../packages.py:185
msgid "An error was encountered while activating your storage configuration."
msgstr ""

#: ../packages.py:353
msgid "Invalid Key"
msgstr "Калиди нодуруст"

#: ../packages.py:354
#, fuzzy
msgid "The key you entered is invalid."
msgstr "Номи воридшуда нодуруст аст."

#: ../packages.py:379 ../rescue.py:248 ../rescue.py:250
#: ../textw/upgrade_text.py:133 ../loader/cdinstall.c:248
#: ../loader/cdinstall.c:251 ../loader/method.c:326
#, fuzzy
msgid "Skip"
msgstr "Гузарондан"

#: ../packages.py:382
msgid "_Skip"
msgstr "_Гузаштан"

#: ../packages.py:414 ../packages.py:437
msgid "Warning! This is pre-release software!"
msgstr ""

#: ../packages.py:415
#, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""

#: ../packages.py:428
msgid "_Install anyway"
msgstr "_Ба ҳар ҳол сабт кунед"

#: ../partIntfHelpers.py:41
#, fuzzy
msgid "Please enter a volume group name."
msgstr "Название группы:"

#: ../partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr ""

#: ../partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr ""

#: ../partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: ../partIntfHelpers.py:63
#, fuzzy
msgid "Please enter a logical volume name."
msgstr "Номи қолибро ворид кунед:"

#: ../partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr ""

#: ../partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr ""

#: ../partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: ../partIntfHelpers.py:101
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: ../partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr ""

#: ../partIntfHelpers.py:121 ../partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr "Нест карда намешавад"

#: ../partIntfHelpers.py:122
#, fuzzy
msgid "You must first select a partition to delete."
msgstr "Сначала следует выделить правило, которое вы хотите удалить."

#: ../partIntfHelpers.py:145 ../partIntfHelpers.py:303
#: ../iw/lvm_dialog_gui.py:747
msgid "Confirm Delete"
msgstr "Тасдиқи несткунӣ"

#: ../partIntfHelpers.py:146
#, fuzzy, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr ""
"Оё шумо дар ҳақиқат ҳамаи\n"
" диаграммаҳоро нобуд сохтан мехоҳед?"

#: ../partIntfHelpers.py:149 ../partIntfHelpers.py:304
#: ../iw/lvm_dialog_gui.py:750 ../iw/lvm_dialog_gui.py:1261
#: ../iw/osbootwidget.py:102 ../iw/partition_gui.py:1333
msgid "_Delete"
msgstr "_Нест кардан"

#: ../partIntfHelpers.py:169
msgid "Notice"
msgstr "Огоҳӣ"

#: ../partIntfHelpers.py:170
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""

#: ../partIntfHelpers.py:185
#, fuzzy
msgid "Format as Swap?"
msgstr "Матн: Иввази ранг"

#: ../partIntfHelpers.py:186
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""

#: ../partIntfHelpers.py:201
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""

#: ../partIntfHelpers.py:206
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""

#: ../partIntfHelpers.py:214
#, fuzzy
msgid "Format?"
msgstr "Андоза"

#: ../partIntfHelpers.py:214 ../iw/partition_gui.py:1034
msgid "_Modify Partition"
msgstr "_Тағйири қисми диск"

#: ../partIntfHelpers.py:214
#, fuzzy
msgid "Do _Not Format"
msgstr "Иваз Накардан"

#: ../partIntfHelpers.py:222
#, fuzzy
msgid "Error with Partitioning"
msgstr "Su вернул ошибку.\n"

#: ../partIntfHelpers.py:223
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""

#: ../partIntfHelpers.py:237
#, fuzzy
msgid "Partitioning Warning"
msgstr "Параметри нодуруст"

#: ../partIntfHelpers.py:238
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: ../partIntfHelpers.py:252 ../iw/partition_gui.py:643
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""

#: ../partIntfHelpers.py:255
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""

#: ../partIntfHelpers.py:261
#, fuzzy
msgid "Format Warning"
msgstr "Параметри нодуруст"

#: ../partIntfHelpers.py:287
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""

#: ../partIntfHelpers.py:291
#, fuzzy, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr ""
"Оё шумо дар ҳақиқат ҳамаи\n"
" диаграммаҳоро нобуд сохтан мехоҳед?"

#: ../partIntfHelpers.py:294
#, fuzzy
msgid "You are about to delete a RAID device."
msgstr ""
"Оё шумо дар ҳақиқат ҳамаи\n"
" диаграммаҳоро нобуд сохтан мехоҳед?"

#: ../partIntfHelpers.py:296
#, fuzzy, python-format
msgid "You are about to delete the %s partition."
msgstr ""
"Оё шумо дар ҳақиқат ҳамаи\n"
" диаграммаҳоро нобуд сохтан мехоҳед?"

#: ../partIntfHelpers.py:300
#, fuzzy, python-format
msgid "You are about to delete the %s %s"
msgstr ""
"Оё шумо дар ҳақиқат ҳамаи\n"
" диаграммаҳоро нобуд сохтан мехоҳед?"

#: ../partIntfHelpers.py:311
msgid "Confirm Reset"
msgstr "Тасдиқи бозсозӣ"

#: ../partIntfHelpers.py:312
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""

#: ../partedUtils.py:108 ../iw/partition_gui.py:779 ../iw/partition_gui.py:927
msgid "None"
msgstr "Ҳеҷ чиз"

#: ../partedUtils.py:170
#, python-format
msgid ""
"The device %s is LDL formatted instead of CDL formatted.  LDL formatted "
"DASDs are not supported for usage during an install of %s.  If you wish to "
"use this disk for installation, it must be re-initialized causing the loss "
"of ALL DATA on this drive.\n"
"\n"
"Would you like to reformat this DASD using CDL format?"
msgstr ""

#: ../partedUtils.py:200
#, python-format
msgid ""
"/dev/%s currently has a %s partition layout.  To use this drive for the "
"installation of %s, it must be re-initialized, causing the loss of ALL DATA "
"on this drive.\n"
"\n"
"Would you like to re-initialize this drive?"
msgstr ""

#: ../partedUtils.py:209 ../storage/devicetree.py:130
msgid "_Ignore drive"
msgstr "_Радкунии дастгоҳ"

#: ../partedUtils.py:210 ../storage/devicetree.py:131
#, fuzzy
msgid "_Re-initialize drive"
msgstr "Танзимоти дастгоҳи лента ҷамъшаванда"

#: ../partedUtils.py:740
msgid "Initializing"
msgstr "Омодасозӣ"

#: ../partedUtils.py:741
#, fuzzy, python-format
msgid "Please wait while formatting drive %s...\n"
msgstr "Интизор шавед то даме, ки ба хидматрасон дархост меравад..."

#: ../partedUtils.py:826
#, python-format
msgid ""
"The partition table on device %s (%s %-0.f MB) was unreadable.\n"
"\n"
"To create new partitions it must be initialized, causing the loss of ALL "
"DATA on this drive.\n"
"\n"
"This operation will override any previous installation choices about which "
"drives to ignore.\n"
"\n"
"Would you like to initialize this drive, erasing ALL DATA?"
msgstr ""

#: ../partedUtils.py:978
#, python-format
msgid ""
"The drive /dev/%s has more than 15 partitions on it.  The SCSI subsystem in "
"the Linux kernel does not allow for more than 15 partitons at this time.  "
"You will not be able to make changes to the partitioning of this disk or use "
"any partitions beyond /dev/%s15 in %s"
msgstr ""

#: ../partedUtils.py:1058 ../storage/__init__.py:851
msgid "No Drives Found"
msgstr "Ягон дастгоҳ ёфт нашуд"

#: ../partedUtils.py:1059 ../storage/__init__.py:852
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""

#: ../platform.py:91 ../platform.py:94 ../platform.py:254 ../platform.py:298
#: ../platform.py:359 ../platform.py:362
#, fuzzy
msgid "Master Boot Record (MBR)"
msgstr "Сабт кардани &маълумоти пурборкардан даруни диск/тақсимот:"

#: ../platform.py:93 ../platform.py:361
#, fuzzy
msgid "First sector of boot partition"
msgstr "Сабт кардани &маълумоти пурборкардан даруни диск/тақсимот:"

#: ../platform.py:165
#, fuzzy
msgid "EFI System Partition"
msgstr "Намуди системаи файлҳо:"

#: ../platform.py:256
#, fuzzy
msgid "PPC PReP Boot"
msgstr "Роҳбари Пурборкардан (LILO)"

#: ../platform.py:300 ../platform.py:303
#, fuzzy
msgid "Apple Bootstrap"
msgstr "Apple RGB"

#: ../rescue.py:167
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: ../rescue.py:174
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: ../rescue.py:193
msgid "Setup Networking"
msgstr "Танзимоти Интернет"

#: ../rescue.py:194
msgid "Do you want to start the network interfaces on this system?"
msgstr ""

#: ../rescue.py:195 ../textw/constants_text.py:56 ../loader/driverdisk.c:483
#: ../loader/driverdisk.c:493 ../loader/hdinstall.c:231
msgid "Yes"
msgstr "Ҳа"

#: ../rescue.py:195 ../rescue.py:197 ../textw/constants_text.py:60
#: ../loader/driverdisk.c:483 ../loader/driverdisk.c:493
msgid "No"
msgstr "Не"

#: ../rescue.py:202
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""

#: ../rescue.py:237 ../rescue.py:306 ../rescue.py:317 ../rescue.py:392
msgid "Rescue"
msgstr "Барқарорсозӣ"

#: ../rescue.py:238
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: ../rescue.py:248 ../iw/partition_gui.py:547 ../storage/__init__.py:135
#: ../storage/devicetree.py:78 ../loader/cdinstall.c:206
#: ../loader/cdinstall.c:214 ../loader/driverdisk.c:444
msgid "Continue"
msgstr "Навбатӣ"

#: ../rescue.py:248 ../rescue.py:253
msgid "Read-Only"
msgstr "Танҳо-Хондан"

#: ../rescue.py:282
msgid "System to Rescue"
msgstr "Барқарорсозии система"

#: ../rescue.py:283
msgid "Which device holds the root partition of your installation?"
msgstr ""

#: ../rescue.py:285 ../rescue.py:289 ../text.py:664 ../text.py:666
msgid "Exit"
msgstr "Хуруҷ"

#: ../rescue.py:307
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:318
#, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:393
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""

#: ../rescue.py:402
msgid "Rescue Mode"
msgstr "Тарзи барқарорсозӣ"

#: ../rescue.py:403
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:416
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: ../text.py:196 ../text.py:280 ../text.py:293 tmp/exnSave.glade.h:7
msgid "Save"
msgstr "Захира"

#: ../text.py:199
msgid "Save to local disk"
msgstr "Ба диски компютерӣ захира кунед"

#: ../text.py:200
#, fuzzy, python-format
msgid "Send to bugzilla (%s)"
msgstr "Ҳеҷ чиз барои Фиристодан нест"

#: ../text.py:201
#, fuzzy
msgid "Send to remote server (scp)"
msgstr "Ба сервер фиристода натавонист.\n"

#: ../text.py:215 ../text.py:228
msgid "User name"
msgstr "Номи корбар"

#: ../text.py:217 ../text.py:230
msgid "Password"
msgstr "Парол"

#: ../text.py:219
#, fuzzy
msgid "Bug Description"
msgstr "Тасвири Қисмат "

#: ../text.py:232
#, fuzzy
msgid "Host (host:port)"
msgstr "Сервер: %1 порт: %2 аллакай вуҷуд дорад."

#: ../text.py:234
#, fuzzy
msgid "Destination file"
msgstr "Файли муайянкунӣ"

#: ../text.py:313
#, fuzzy
msgid "Passphrase for encrypted device"
msgstr "Барраси барои дастгоҳи навор."

#: ../text.py:326
msgid "Also add this passphrase to all existing encrypted devices"
msgstr ""

#: ../text.py:357
#, python-format
msgid "The passphrase must be at least %d characters long."
msgstr ""

#: ../text.py:387 tmp/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr "Гузарвожа"

#: ../text.py:395 tmp/lukspassphrase.glade.h:7
#, fuzzy
msgid "This is a global passphrase"
msgstr "Сигнали номуайян"

#: ../text.py:461 ../text.py:465
msgid "Repository editing is not available in text mode."
msgstr ""

#: ../text.py:505
#, fuzzy, python-format
msgid "Please enter your %(instkey)s"
msgstr "Введите ваш текущий пароль:"

#: ../text.py:522 tmp/instkey.glade.h:6
#, fuzzy, no-c-format, python-format
msgid "Skip entering %(instkey)s"
msgstr "Радкунии &саҳифабанд"

#: ../text.py:570 ../loader/lang.c:63 ../loader/loader.c:215
#, c-format, python-format
msgid "Welcome to %s for %s"
msgstr "Марҳамат намоед ба %s барои %s"

#: ../text.py:572
#, python-format
msgid "Welcome to %s"
msgstr "Марҳамат намоед ба %s"

#: ../text.py:574
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""

#: ../text.py:664 ../loader/net.c:111 ../loader/net.c:454 ../loader/net.c:510
#: ../loader/net.c:655 ../loader/net.c:663 ../loader/net.c:1086
#: ../loader/net.c:1093
msgid "Retry"
msgstr "Кӯшиши нав"

#: ../text.py:690
msgid "Cancelled"
msgstr "Бекор карда шуд"

#: ../text.py:691
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""

#: ../upgrade.py:92 ../upgrade.py:113
#, fuzzy
msgid "Proceed with upgrade?"
msgstr "Кушодан бо Kaffeine"

#: ../upgrade.py:93
#, python-format
msgid ""
"You have choosen the upgrade for %s architecture, but the installed system "
"is for %s architecture. \n"
"\n"
msgstr ""

#: ../upgrade.py:97
#, python-format
msgid "Would you like to upgrade  the installed system to the %s architecture?"
msgstr ""

#: ../upgrade.py:114
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""

#: ../upgrade.py:118
msgid "Would you like to continue with the upgrade?"
msgstr ""

#: ../upgrade.py:239
msgid "Mount failed"
msgstr "Васлкунӣ қатъ карда шуд"

#: ../upgrade.py:240
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""

#: ../upgrade.py:247
msgid "Upgrade root not found"
msgstr ""

#: ../upgrade.py:248
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""

#: ../upgrade.py:271
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""

#: ../upgrade.py:277
#, fuzzy
msgid "Absolute Symlinks"
msgstr "6: Аҳамияти комилӣ"

#: ../upgrade.py:288
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""

#: ../upgrade.py:294
msgid "Invalid Directories"
msgstr "Феҳрастҳои нодуруст"

#: ../upgrade.py:299
#, python-format
msgid "%s not found"
msgstr "%s ёфт нашуд"

#: ../vnc.py:136
#, fuzzy, python-format
msgid "%s %s installation on host %s"
msgstr "Сеанс X Windows на удалённом узле"

#: ../vnc.py:138
#, python-format
msgid "%s %s installation"
msgstr "Сабткунии %s %s"

#: ../vnc.py:165
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: ../vnc.py:179
msgid "Connected!"
msgstr "Пайваст карда шуд!"

#: ../vnc.py:182
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: ../vnc.py:188
#, python-format
msgid "Giving up attempting to connect after %d tries!\n"
msgstr ""

#: ../vnc.py:197
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: ../vnc.py:199
msgid "Please manually connect your vnc client to begin the install."
msgstr ""

#: ../vnc.py:202
msgid "Starting VNC..."
msgstr "Оғозкунии VNC..."

#: ../vnc.py:227
msgid "The VNC server is now running."
msgstr ""

#: ../vnc.py:240
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: ../vnc.py:245
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: ../vnc.py:249
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: ../vnc.py:251
#, fuzzy
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr "Сервери %1: Сар додани пайвастагӣ."

#: ../vnc.py:273 ../vnc.py:366
msgid "VNC Configuration"
msgstr "Танзимоти VNC"

#: ../vnc.py:276 ../vnc.py:370
msgid "No password"
msgstr "Бе парол"

#: ../vnc.py:278 ../vnc.py:373
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""

#: ../vnc.py:286 ../vnc.py:381 ../textw/userauth_text.py:47
msgid "Password:"
msgstr "Парол:"

#: ../vnc.py:287 ../vnc.py:382 ../textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "Парол (тасдиқ):"

#: ../vnc.py:305 ../vnc.py:404 ../textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "Пароли нодуруст"

#: ../vnc.py:306 ../vnc.py:405 ../textw/userauth_text.py:71
#, fuzzy
msgid "The passwords you entered were different. Please try again."
msgstr "Паролҳо баробар наомаданд. Лутфан, боз кӯшиш кунед."

#: ../vnc.py:311 ../vnc.py:410 ../textw/userauth_text.py:66
msgid "Password Length"
msgstr "Дарозии парол"

#: ../vnc.py:312 ../vnc.py:411
msgid "The password must be at least six characters long."
msgstr ""

#: ../vnc.py:334
#, fuzzy
msgid "Unable to Start X"
msgstr "Барномаи ksplash-ро иҷро кардан натавонист."

#: ../vnc.py:336
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""

#: ../vnc.py:355
msgid "Start VNC"
msgstr "Оғози VNC"

#: ../vnc.py:356 ../vnc.py:358
msgid "Use text mode"
msgstr "Ҳолати матниро истифода баред"

#: ../yuminstall.py:79
#, python-format
msgid "%s MB"
msgstr "%s МБ"

#: ../yuminstall.py:82
#, python-format
msgid "%s KB"
msgstr "%s КБ"

#: ../yuminstall.py:85
#, python-format
msgid "%s Byte"
msgstr "%s Бит"

#: ../yuminstall.py:87
#, python-format
msgid "%s Bytes"
msgstr "%s Бит"

#: ../yuminstall.py:131
#, fuzzy
msgid "Processing"
msgstr "Коркард "

#: ../yuminstall.py:132
msgid "Preparing transaction from installation source..."
msgstr ""

#: ../yuminstall.py:163
#, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "<b>Сабткунии %s</b> (%s)\n"

#: ../yuminstall.py:212
#, fuzzy, python-format
msgid "%s of %s packages completed"
msgstr "Хотима дода нашуд"

#: ../yuminstall.py:222
#, fuzzy
msgid "Finishing upgrade"
msgstr "1MB Зиёдкунӣ"

#: ../yuminstall.py:223
msgid "Finishing upgrade process.  This may take a little while..."
msgstr ""

#: ../yuminstall.py:293 ../iw/task_gui.py:301
msgid "Error Setting Up Repository"
msgstr "Ҳангоми танзими феҳраст хато пайдо шуд"

#: ../yuminstall.py:294 ../iw/task_gui.py:302
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%s\n"
"\n"
"Please provide the correct information for installing %s."
msgstr ""

#: ../yuminstall.py:353
msgid "Change Disc"
msgstr "Ивази диск"

#: ../yuminstall.py:354
#, python-format
msgid "Please insert %s disc %d to continue."
msgstr ""

#: ../yuminstall.py:364
msgid "Wrong Disc"
msgstr "Диски нодуруст"

#: ../yuminstall.py:365
#, python-format
msgid "That's not the correct %s disc."
msgstr ""

#: ../yuminstall.py:372
#, fuzzy
msgid "Unable to access the disc."
msgstr "Идоракунии дастрасӣ ба репозиторияҳо"

#: ../yuminstall.py:530
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""

#: ../yuminstall.py:654 ../yuminstall.py:656
msgid "Re_boot"
msgstr "Боз_худборшавӣ"

#: ../yuminstall.py:654
msgid "_Eject"
msgstr "_Баровардани диск"

#: ../yuminstall.py:665
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: ../yuminstall.py:708
msgid "Retrying"
msgstr "Аз нав кӯшиш мекунад"

#: ../yuminstall.py:708
#, fuzzy
msgid "Retrying download..."
msgstr "Фақат фаровир"

#: ../yuminstall.py:780
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""

#: ../yuminstall.py:815 ../yuminstall.py:816
#, fuzzy
msgid "file conflicts"
msgstr "Нусхабардории файл"

#: ../yuminstall.py:817
#, fuzzy
msgid "older package(s)"
msgstr "Сабткунии қуттиҳо"

#: ../yuminstall.py:818
#, fuzzy
msgid "insufficient disk space"
msgstr "Андозаи озод дар диск:"

#: ../yuminstall.py:819
#, fuzzy
msgid "insufficient disk inodes"
msgstr "Захира дар диск"

#: ../yuminstall.py:820
#, fuzzy
msgid "package conflicts"
msgstr "Сабткунии қуттиҳо"

#: ../yuminstall.py:821
#, fuzzy
msgid "package already installed"
msgstr "Настройка установленных шрифтов..."

#: ../yuminstall.py:822
#, fuzzy
msgid "required package"
msgstr "Сабткунии қуттиҳо"

#: ../yuminstall.py:823
#, fuzzy
msgid "package for incorrect arch"
msgstr "Сардодан бо рӯйхати хатоҳо барои <package>"

#: ../yuminstall.py:824
#, fuzzy
msgid "package for incorrect os"
msgstr "Сардодан бо рӯйхати хатоҳо барои <package>"

#: ../yuminstall.py:838
msgid "You need more space on the following file systems:\n"
msgstr ""

#: ../yuminstall.py:851
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""

#: ../yuminstall.py:854
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr ""

#: ../yuminstall.py:861 ../yuminstall.py:866
#, fuzzy
msgid "Error Running Transaction"
msgstr "Хатоӣ дар ба амалории созиш"

#: ../yuminstall.py:1023
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""

#: ../yuminstall.py:1057
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: ../yuminstall.py:1095
#, fuzzy
msgid "Retrieving installation information..."
msgstr "Васл кардан. Аҳбороти борои ҷадвал гирифтан"

#: ../yuminstall.py:1097
#, fuzzy, python-format
msgid "Retrieving installation information for %s..."
msgstr "Гузарвожа барои фаровири иттилоот оиди вақти машғулӣ ва вақти холӣ"

#: ../yuminstall.py:1099
msgid "Installation Progress"
msgstr "Ҳолати сабткунӣ"

#: ../yuminstall.py:1112 ../textw/constants_text.py:64
msgid "Edit"
msgstr "Тағйир"

#: ../yuminstall.py:1125 ../yuminstall.py:1313
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""

#: ../yuminstall.py:1176 ../iw/GroupSelector.py:485
msgid "Uncategorized"
msgstr ""

#: ../yuminstall.py:1338
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""

#: ../yuminstall.py:1357
#, fuzzy
msgid "Reboot?"
msgstr "перезагрузить"

#: ../yuminstall.py:1358
msgid "The system will be rebooted now."
msgstr ""

#: ../yuminstall.py:1511
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: ../yuminstall.py:1540
#, python-format
msgid ""
"The arch of the release of %s you are upgrading to appears to be %s which "
"does not match your previously installed arch of %s.  This is likely to not "
"succeed.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: ../yuminstall.py:1591
#, fuzzy
msgid "Post Upgrade"
msgstr "1MB Зиёдкунӣ"

#: ../yuminstall.py:1592
msgid "Performing post upgrade configuration..."
msgstr ""

#: ../yuminstall.py:1594
#, fuzzy
msgid "Post Install"
msgstr "Сабт кардани &қайд гирӣ"

#: ../yuminstall.py:1595
msgid "Performing post install configuration..."
msgstr ""

#: ../yuminstall.py:1820
msgid "Install Starting"
msgstr "Оғози сабткунӣ"

#: ../yuminstall.py:1821
msgid "Starting install process.  This may take several minutes..."
msgstr ""

#: ../yuminstall.py:1859
#, fuzzy
msgid "Dependency Check"
msgstr "Dependency Check"

#: ../yuminstall.py:1860
msgid "Checking dependencies in packages selected for installation..."
msgstr ""

#: ../iw/GroupSelector.py:147
#, fuzzy, python-format
msgid "Packages in %s"
msgstr "Калон кардан"

#: ../iw/GroupSelector.py:424
#, fuzzy, python-format
msgid "Optional packages selected: %d of %d"
msgstr "Дар пайраҳаи ивазо бурида шудааст "

#: ../iw/GroupSelector.py:425
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: ../iw/account_gui.py:52
msgid "Root _Password:"
msgstr " _Пароли Root:"

#: ../iw/account_gui.py:54
msgid "_Confirm:"
msgstr "_Тасдиқ кунед:"

#: ../iw/account_gui.py:97
msgid "Caps Lock is on."
msgstr "Caps Lock фаъол аст."

#: ../iw/account_gui.py:107 ../iw/account_gui.py:115 ../iw/account_gui.py:122
#: ../iw/account_gui.py:144 ../textw/userauth_text.py:74
#, fuzzy
msgid "Error with Password"
msgstr "Su вернул ошибку.\n"

#: ../iw/account_gui.py:108
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: ../iw/account_gui.py:116
#, fuzzy
msgid "The passwords you entered were different.  Please try again."
msgstr "Паролҳо баробар наомаданд. Лутфан, боз кӯшиш кунед."

#: ../iw/account_gui.py:123
msgid "The root password must be at least six characters long."
msgstr ""

#: ../iw/account_gui.py:132 ../textw/userauth_text.py:83
msgid "Weak Password"
msgstr "Пароли осон"

#: ../iw/account_gui.py:133 ../textw/userauth_text.py:84
#, python-format
msgid ""
"Weak password provided: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: ../iw/account_gui.py:145 ../textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""

#: ../iw/autopart_type.py:101
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""

#: ../iw/autopart_type.py:124
#, fuzzy
msgid "Resize FileSystem Error"
msgstr "Ба ҳамон системаи файлӣ қарор гирифтан"

#: ../iw/autopart_type.py:125 ../iw/autopart_type.py:134
#, python-format
msgid "%s: %s"
msgstr ""

#: ../iw/autopart_type.py:133
msgid "Resize Device Error"
msgstr ""

#: ../iw/autopart_type.py:208
msgid ""
"Do you really want to boot from a disk which is not used for installation?"
msgstr ""

#: ../iw/autopart_type.py:285
#, fuzzy
msgid "Invalid Initiator Name"
msgstr "Номи Нодурусти Ҷойгир"

#: ../iw/autopart_type.py:286
#, fuzzy
msgid "You must provide an initiator name."
msgstr "Шумо бояд номи нохолиро пешкаш намоед."

#: ../iw/autopart_type.py:321
#, fuzzy
msgid "Error with Data"
msgstr "Ошибка четности данных"

#: ../iw/autopart_type.py:398 ../iw/autopart_type.py:399
msgid "Rescanning disks"
msgstr ""

#: ../iw/autopart_type.py:452 ../textw/partition_text.py:63
#, fuzzy
msgid "Use entire drive"
msgstr "_Радкунии дастгоҳ"

#: ../iw/autopart_type.py:453 ../textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr ""

#: ../iw/autopart_type.py:454
#, fuzzy
msgid "Shrink current system"
msgstr "_Рамзгузории система"

#: ../iw/autopart_type.py:455 ../textw/partition_text.py:65
#, fuzzy
msgid "Use free space"
msgstr "Ҷои холӣ (МБ)"

#: ../iw/autopart_type.py:456
#, fuzzy
msgid "Create custom layout"
msgstr "Эҷоди нақша қатъ карда шуд."

#: ../iw/blpasswidget.py:47
#, fuzzy
msgid "_Use a boot loader password"
msgstr "Гузарвожае, ки истифода бурдан мехоҳед."

#: ../iw/blpasswidget.py:48
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""

#: ../iw/blpasswidget.py:79
msgid "Change _password"
msgstr "Ивази _парол"

#: ../iw/blpasswidget.py:102
#, fuzzy
msgid "Enter Boot Loader Password"
msgstr "Калимаи рамзи нави гувоҳномаро ворид кунед"

#: ../iw/blpasswidget.py:108
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""

#: ../iw/blpasswidget.py:115
msgid "_Password:"
msgstr "_Парол:"

#: ../iw/blpasswidget.py:121
msgid "Con_firm:"
msgstr "Тас_диқ кунед:"

#: ../iw/blpasswidget.py:142
#, fuzzy
msgid "Passwords don't match"
msgstr "Паролҳо баробар наомаданд"

#: ../iw/blpasswidget.py:143
#, fuzzy
msgid "Passwords do not match"
msgstr "Паролҳо баробар наомаданд"

#: ../iw/blpasswidget.py:152
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: ../iw/bootloader_main_gui.py:36
#, fuzzy
msgid "Boot Loader Configuration"
msgstr "Танзимоти Интернет барои %s"

#: ../iw/bootloader_main_gui.py:179 ../iw/bootloader_main_gui.py:184
#: ../iw/bootloader_main_gui.py:221
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr ""

#: ../iw/bootloader_main_gui.py:227
msgid "_Change device"
msgstr "_Ивази дастгоҳ"

#: ../iw/congrats_gui.py:33
#, fuzzy
msgid "Congratulations"
msgstr "Табрик!"

#: ../iw/congrats_gui.py:74 ../textw/complete_text.py:39
#, fuzzy, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr "Ҳангоми нопурра будани ахборот, кӯшиш кунед"

#: ../iw/congrats_gui.py:77 ../textw/complete_text.py:42
msgid "Shutdown"
msgstr ""

#: ../iw/congrats_gui.py:79 ../textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr ""

#: ../iw/congrats_gui.py:81 ../textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr ""

#: ../iw/congrats_gui.py:86 ../textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""

#: ../iw/congrats_gui.py:90 ../textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""

#: ../iw/examine_gui.py:37
#, fuzzy
msgid "Upgrade Examine"
msgstr "1MB Зиёдкунӣ"

#: ../iw/examine_gui.py:59
#, python-format
msgid "_Install %s"
msgstr "_Сабт кардани %s"

#: ../iw/examine_gui.py:61
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""

#: ../iw/examine_gui.py:65
#, fuzzy
msgid "_Upgrade an existing installation"
msgstr "Оғози сабткунии графикӣ..."

#: ../iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""

#: ../iw/examine_gui.py:116
msgid "The following installed system will be upgraded:"
msgstr ""

#: ../iw/examine_gui.py:129
msgid "Unknown Linux system"
msgstr "Системаи Linux-и номайян"

#: ../iw/language_gui.py:33 ../textw/language_text.py:46
msgid "Language Selection"
msgstr "Интихоби забонҳо"

#: ../iw/language_gui.py:81 ../textw/language_text.py:47 ../loader/lang.c:376
msgid "What language would you like to use during the installation process?"
msgstr "Кадом забонро Шумо мехоҳед барои сабткунии система истифода мебаред?"

#: ../iw/lvm_dialog_gui.py:120 ../iw/lvm_dialog_gui.py:166
#: ../iw/lvm_dialog_gui.py:178 ../iw/lvm_dialog_gui.py:217
#: ../iw/lvm_dialog_gui.py:293 ../iw/lvm_dialog_gui.py:613
#, fuzzy
msgid "Not enough space"
msgstr "Ҷой дар диск нокифоя аст?"

#: ../iw/lvm_dialog_gui.py:121
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""

#: ../iw/lvm_dialog_gui.py:130
#, fuzzy
msgid "Confirm Physical Extent Change"
msgstr "Тасдиқи Тағироти Гузоришҳои Экран"

#: ../iw/lvm_dialog_gui.py:131
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""

#: ../iw/lvm_dialog_gui.py:140 ../iw/lvm_dialog_gui.py:200
msgid "C_ontinue"
msgstr "Д_авом диҳед"

#: ../iw/lvm_dialog_gui.py:167
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the "
"volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:179
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is too large compared to the size of the smallest physical volume "
"(%10.2f MB) in the volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:193
msgid "Too small"
msgstr "Хеле хурд"

#: ../iw/lvm_dialog_gui.py:194
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:218
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""

#: ../iw/lvm_dialog_gui.py:294
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""

#: ../iw/lvm_dialog_gui.py:371
#, fuzzy
msgid "Make Logical Volume"
msgstr "Нишон додани Панели &Баландии овоз"

#: ../iw/lvm_dialog_gui.py:373
#, fuzzy, python-format
msgid "Edit Logical Volume: %s"
msgstr "Нишон додани Панели &Баландии овоз"

#: ../iw/lvm_dialog_gui.py:407 ../iw/partition_dialog_gui.py:306
#: ../iw/raid_dialog_gui.py:364
msgid "_Mount Point:"
msgstr "_Нуқтаи васл:"

#: ../iw/lvm_dialog_gui.py:415 ../iw/raid_dialog_gui.py:373
msgid "_File System Type:"
msgstr "_Намуди системаи файлҳо:"

#: ../iw/lvm_dialog_gui.py:425 ../iw/raid_dialog_gui.py:383
#, fuzzy
msgid "Original File System Type:"
msgstr "Эҷоди намунаи номи файли нав %1"

#: ../iw/lvm_dialog_gui.py:430 ../iw/raid_dialog_gui.py:388
msgid "Unknown"
msgstr "Номуайян"

#: ../iw/lvm_dialog_gui.py:436 ../iw/partition_dialog_gui.py:351
#: ../iw/raid_dialog_gui.py:394
#, fuzzy
msgid "Original File System Label:"
msgstr "Танҳо системаи файлиро хонед"

#: ../iw/lvm_dialog_gui.py:444
#, fuzzy
msgid "_Logical Volume Name:"
msgstr "Номи хидматгоҳи NFS:"

#: ../iw/lvm_dialog_gui.py:452
#, fuzzy
msgid "Logical Volume Name:"
msgstr "Номи хидматгоҳи NFS:"

#: ../iw/lvm_dialog_gui.py:460 ../iw/partition_dialog_gui.py:361
msgid "_Size (MB):"
msgstr "_Андоза (МБ):"

#: ../iw/lvm_dialog_gui.py:465
msgid "Size (MB):"
msgstr "Андоза (МБ):"

#: ../iw/lvm_dialog_gui.py:474
#, fuzzy, python-format
msgid "(Max size is %s MB)"
msgstr "<tr><td><i>Андозаи қуттиҳои ниҳоят</i></td><td>%1</td></tr>"

#: ../iw/lvm_dialog_gui.py:495 ../iw/partition_dialog_gui.py:413
#: ../iw/partition_ui_helpers_gui.py:322 ../iw/raid_dialog_gui.py:345
msgid "_Encrypt"
msgstr "_Рамзгузорӣ"

#: ../iw/lvm_dialog_gui.py:540
msgid "Illegal Logical Volume Name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:555
msgid "Illegal logical volume name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:556
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:584
msgid "Mount point in use"
msgstr "Нуқтаи васл банд аст"

#: ../iw/lvm_dialog_gui.py:585
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:600
msgid "Illegal size"
msgstr "Андозаи нодуруст"

#: ../iw/lvm_dialog_gui.py:601
msgid "The requested size as entered is not a valid number greater than 0."
msgstr ""

#: ../iw/lvm_dialog_gui.py:614
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). To increase this limit you can create more Physical "
"Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:708
#, fuzzy
msgid "No free slots"
msgstr "Пайвасти Сигналҳо /Такягоҳо"

#: ../iw/lvm_dialog_gui.py:709
#, python-format
msgid "You cannot create more than %s logical volumes per volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:715
#, fuzzy
msgid "No free space"
msgstr "Ҷои холӣ (МБ)"

#: ../iw/lvm_dialog_gui.py:716
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""

#: ../iw/lvm_dialog_gui.py:748
#, fuzzy, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "Дар ҳақиқат мехоҳед, ки тахаллусҳои интихобкардаро нобуд созед?"

#: ../iw/lvm_dialog_gui.py:861
#, fuzzy
msgid "Invalid Volume Group Name"
msgstr "Номи гурӯҳи навро ворид кунед:"

#: ../iw/lvm_dialog_gui.py:870
#, fuzzy
msgid "Name in use"
msgstr "Истифода баред номи рамзҳоро"

#: ../iw/lvm_dialog_gui.py:871
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:1095
#, fuzzy
msgid "Not enough physical volumes"
msgstr "Хатогии Наҳвӣ: Нишонвандҳо нокифояанд\n"

#: ../iw/lvm_dialog_gui.py:1096
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""

#: ../iw/lvm_dialog_gui.py:1107
msgid "Make LVM Volume Group"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1110
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1112
msgid "Edit LVM Volume Group"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1128
#, fuzzy
msgid "_Volume Group Name:"
msgstr "&Номи гурӯҳи амалиёт:"

#: ../iw/lvm_dialog_gui.py:1136
#, fuzzy
msgid "Volume Group Name:"
msgstr "&Номи гурӯҳи амалиёт:"

#: ../iw/lvm_dialog_gui.py:1144
#, fuzzy
msgid "_Physical Extent:"
msgstr "Хотири Ҷисмонӣ"

#: ../iw/lvm_dialog_gui.py:1159
#, fuzzy
msgid "Physical Volumes to _Use:"
msgstr "Суроғаи хато мавриди истифода"

#: ../iw/lvm_dialog_gui.py:1165
msgid "Used Space:"
msgstr "Ҷои истифодашуда:"

#: ../iw/lvm_dialog_gui.py:1182
msgid "Free Space:"
msgstr "Ҷои холӣ:"

#: ../iw/lvm_dialog_gui.py:1200
msgid "Total Space:"
msgstr "Ҳамагӣ:"

#: ../iw/lvm_dialog_gui.py:1235
#, fuzzy
msgid "Logical Volume Name"
msgstr "Номи хидматгоҳи NFS:"

#: ../iw/lvm_dialog_gui.py:1238 ../iw/partition_gui.py:366
#: ../iw/upgrade_swap_gui.py:140 ../textw/upgrade_text.py:118
msgid "Mount Point"
msgstr "Нуқтаи васл"

#: ../iw/lvm_dialog_gui.py:1241 ../iw/partition_gui.py:371
msgid "Size (MB)"
msgstr "Андоза (МБ)"

#: ../iw/lvm_dialog_gui.py:1255 ../iw/osbootwidget.py:94
msgid "_Add"
msgstr "_Илова"

#: ../iw/lvm_dialog_gui.py:1258 ../iw/osbootwidget.py:98
#: ../iw/partition_gui.py:1332
msgid "_Edit"
msgstr "_Таҳрир"

#: ../iw/lvm_dialog_gui.py:1273
#, fuzzy
msgid "_Logical Volumes"
msgstr "Пешниҳодҳои мантиқӣ"

#: ../iw/netconfig_dialog.py:193 ../textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""

#: ../iw/netconfig_dialog.py:195 ../iw/netconfig_dialog.py:204
#: ../textw/netconfig_text.py:35 ../textw/netconfig_text.py:40
#, fuzzy
msgid "Error With Data"
msgstr "Ошибка четности данных"

#: ../iw/netconfig_dialog.py:203
#, fuzzy, python-format
msgid "A value is required for the field %s."
msgstr "Маънои\"%1\" барои драйвер аввалиндараҷа нест."

#: ../iw/netconfig_dialog.py:213
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr ""

#: ../iw/netconfig_dialog.py:215
#, fuzzy
msgid "Error Configuring Network"
msgstr "Хатои дохилӣ: Дастгоҳ танзим нашудааст."

#: ../iw/netconfig_dialog.py:237
#, fuzzy
msgid "Dynamic IP"
msgstr "Адреси IP Динамикӣ"

#: ../iw/netconfig_dialog.py:238
#, python-format
msgid "Sending request for IP information for %s..."
msgstr ""

#: ../iw/netconfig_dialog.py:253 ../iw/netconfig_dialog.py:256
#: ../textw/netconfig_text.py:214 ../textw/netconfig_text.py:217
msgid "IP Address"
msgstr "Суроғаи IP"

#: ../iw/netconfig_dialog.py:263 ../textw/netconfig_text.py:224
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr ""

#: ../iw/netconfig_dialog.py:264 ../iw/netconfig_dialog.py:270
#: ../iw/netconfig_dialog.py:278 ../iw/netconfig_dialog.py:281
#: ../textw/netconfig_text.py:225 ../textw/netconfig_text.py:231
#: ../textw/netconfig_text.py:239
#, fuzzy
msgid "IPv4 Network Mask"
msgstr "Ниқоби бехатарии феҳрист:"

#: ../iw/netconfig_dialog.py:291 ../textw/netconfig_text.py:252
msgid "Gateway"
msgstr "Суроҳаи хидматгоҳ"

#: ../iw/netconfig_dialog.py:301 ../textw/netconfig_text.py:262
msgid "Nameserver"
msgstr "Номи хидматгоҳ"

#: ../iw/netconfig_dialog.py:310
#, fuzzy
msgid "Error configuring network device:"
msgstr "Хатои дохилӣ: Дастгоҳ танзим нашудааст."

#: ../iw/network_gui.py:60 ../iw/network_gui.py:66
#, fuzzy
msgid "Error with Hostname"
msgstr "Su вернул ошибку.\n"

#: ../iw/network_gui.py:61
msgid "You must enter a valid hostname for this computer."
msgstr ""

#: ../iw/network_gui.py:67
#, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""

#: ../iw/osbootwidget.py:49
msgid "Boot loader operating system list"
msgstr ""

#: ../iw/osbootwidget.py:65
msgid "Default"
msgstr "Стандартӣ"

#: ../iw/osbootwidget.py:65 ../iw/partition_gui.py:365
msgid "Label"
msgstr "Тамға"

#: ../iw/osbootwidget.py:65 ../iw/partition_gui.py:364
msgid "Device"
msgstr "Дастгоҳ"

#: ../iw/osbootwidget.py:128
#, fuzzy
msgid "Image"
msgstr "Тасвир"

#: ../iw/osbootwidget.py:135
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""

#: ../iw/osbootwidget.py:143
msgid "_Label"
msgstr "_Тамға"

#: ../iw/osbootwidget.py:151
msgid "_Device"
msgstr "_Дастгоҳ"

#: ../iw/osbootwidget.py:181
#, fuzzy
msgid "Default Boot _Target"
msgstr "Пурбор кардани ядро/СО ба пешфарз &баъди:"

#: ../iw/osbootwidget.py:210
msgid "You must specify a label for the entry"
msgstr ""

#: ../iw/osbootwidget.py:219
msgid "Boot label contains illegal characters"
msgstr ""

#: ../iw/osbootwidget.py:243
#, fuzzy
msgid "Duplicate Label"
msgstr "Нишонаҳои Атомҳо"

#: ../iw/osbootwidget.py:244
msgid "This label is already in use for another boot entry."
msgstr ""

#: ../iw/osbootwidget.py:257
msgid "Duplicate Device"
msgstr "Нусхабардории дастгоҳ"

#: ../iw/osbootwidget.py:258
msgid "This device is already being used for another boot entry."
msgstr ""

#: ../iw/osbootwidget.py:322
msgid "Cannot Delete"
msgstr "Нест карда намешавад"

#: ../iw/osbootwidget.py:323
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""

#: ../iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr "Интихобҳои иловагии андоза"

#: ../iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr "_Андозаи муайяншуда"

#: ../iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr ""

#: ../iw/partition_dialog_gui.py:74
msgid "Fill to maximum _allowable size"
msgstr ""

#: ../iw/partition_dialog_gui.py:276
msgid "Add Partition"
msgstr "Иловаи қисми диск"

#: ../iw/partition_dialog_gui.py:278
#, fuzzy, python-format
msgid "Edit Partition: %s"
msgstr "Таҳрири қисми диск: /dev/%s"

#: ../iw/partition_dialog_gui.py:315
msgid "File System _Type:"
msgstr "Намуди _системаи файлҳо:"

#: ../iw/partition_dialog_gui.py:331
msgid "Allowable _Drives:"
msgstr "Дастгоҳҳои _иҷозашуда:"

#: ../iw/partition_dialog_gui.py:400
msgid "Force to be a _primary partition"
msgstr ""

#: ../iw/partition_gui.py:317
#, fuzzy, python-format
msgid "Drive %s (Geom: %s/%s/%s) (Model: %s)"
msgstr "Интихоби Шакли Чопгар"

#: ../iw/partition_gui.py:322
#, fuzzy, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr "Интихоби Шакли Чопгар"

#: ../iw/partition_gui.py:367
msgid "Type"
msgstr "Намуд"

#: ../iw/partition_gui.py:370
#, fuzzy
msgid "Format"
msgstr "Андоза"

#: ../iw/partition_gui.py:408
#, fuzzy
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr "Нуқтаи монтаж кунӣ (/mnt/floppy):"

#: ../iw/partition_gui.py:410
msgid ""
"Size\n"
"(MB)"
msgstr ""
"Андоза\n"
"(MB)"

#: ../iw/partition_gui.py:522
#, fuzzy
msgid "Partitioning"
msgstr "Ҳолати бахшбандӣ"

#: ../iw/partition_gui.py:612
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: ../iw/partition_gui.py:614
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""

#: ../iw/partition_gui.py:620
#, fuzzy
msgid "Partitioning Errors"
msgstr "Гузориши хатогиҳо"

#: ../iw/partition_gui.py:627
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: ../iw/partition_gui.py:629
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: ../iw/partition_gui.py:634
#, fuzzy
msgid "Partitioning Warnings"
msgstr "Ҳолати бахшбандӣ"

#: ../iw/partition_gui.py:655
#, fuzzy
msgid "Format Warnings"
msgstr "Ба андоза даровардани лента..."

#: ../iw/partition_gui.py:660
msgid "_Format"
msgstr "_Андоза"

#: ../iw/partition_gui.py:686
#, fuzzy
msgid "LVM Volume Groups"
msgstr "Гурӯҳҳои шахсии корванд"

#: ../iw/partition_gui.py:733
msgid "RAID Devices"
msgstr "Дастгоҳҳои RAID"

#: ../iw/partition_gui.py:800 ../loader/hdinstall.c:231
msgid "Hard Drives"
msgstr "Дискҳои компютерӣ"

#: ../iw/partition_gui.py:895
msgid "Extended"
msgstr "Васеъшуда"

#: ../iw/partition_gui.py:897
msgid "software RAID"
msgstr "нармафзори RAID"

#: ../iw/partition_gui.py:929
msgid "Free"
msgstr "Холӣ"

#: ../iw/partition_gui.py:1022 ../storage/partitioning.py:174
#: ../storage/partitioning.py:207
#, fuzzy
msgid "Error Partitioning"
msgstr "Ҳолати бахшбандӣ"

#: ../iw/partition_gui.py:1023
#, fuzzy, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "Амалиёти дархостшуда иҷро карда намешавад"

#: ../iw/partition_gui.py:1032
#, python-format
msgid "Warning: %s."
msgstr "Огоҳӣ: %s."

#: ../iw/partition_gui.py:1062 ../iw/partition_gui.py:1069
msgid "Unable To Edit"
msgstr "Таҳрир карда намешавад"

#: ../iw/partition_gui.py:1063
#, fuzzy
msgid "You must select a device to edit"
msgstr "Сначала следует выделить правило, которое вы хотите удалить."

#: ../iw/partition_gui.py:1070
#, fuzzy, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr "Ин файлҳои зиёдро таҳрир мекунад. Шумо боварӣ доред?"

#: ../iw/partition_gui.py:1177 ../iw/partition_gui.py:1189
msgid "Not supported"
msgstr "Пуштибонӣ намешавад"

#: ../iw/partition_gui.py:1178
msgid "LVM is NOT supported on this platform."
msgstr ""

#: ../iw/partition_gui.py:1190
msgid "Software RAID is NOT supported on this platform."
msgstr ""

#: ../iw/partition_gui.py:1197
msgid "No RAID minor device numbers available"
msgstr ""

#: ../iw/partition_gui.py:1198
msgid ""
"A software RAID device cannot be created because all of the available RAID "
"minor device numbers have been used."
msgstr ""

#: ../iw/partition_gui.py:1210
msgid "RAID Options"
msgstr "Интихобҳои RAID"

#: ../iw/partition_gui.py:1221
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
"\n"
"You currently have %s software RAID partition(s) free to use.\n"
"\n"
msgstr ""

#: ../iw/partition_gui.py:1232
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""

#: ../iw/partition_gui.py:1238
#, fuzzy
msgid "What do you want to do now?"
msgstr ""
"Шумо аз рости мехоҳед ,кучонед :\n"
" %1?"

#: ../iw/partition_gui.py:1247
msgid "Create a software RAID _partition."
msgstr ""

#: ../iw/partition_gui.py:1250
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr ""

#: ../iw/partition_gui.py:1254
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr ""

#: ../iw/partition_gui.py:1296
msgid "Couldn't Create Drive Clone Editor"
msgstr ""

#: ../iw/partition_gui.py:1297
msgid "The drive clone editor could not be created for some reason."
msgstr ""

#: ../iw/partition_gui.py:1331
msgid "Ne_w"
msgstr "На_в"

#: ../iw/partition_gui.py:1334
msgid "Re_set"
msgstr "Боз_созӣ"

#: ../iw/partition_gui.py:1335
msgid "R_AID"
msgstr "R_AID"

#: ../iw/partition_gui.py:1336
msgid "_LVM"
msgstr "_LVM"

#: ../iw/partition_gui.py:1377
msgid "Hide RAID device/LVM Volume _Group members"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:106 ../iw/partition_ui_helpers_gui.py:128
#: ../iw/partition_ui_helpers_gui.py:130
msgid "<Not Applicable>"
msgstr "<Истифоданашаванда>"

#: ../iw/partition_ui_helpers_gui.py:306
#, fuzzy
msgid "_Format as:"
msgstr "Додани худро ҳамчун"

#: ../iw/partition_ui_helpers_gui.py:325
msgid "Mi_grate filesystem to:"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:351 tmp/autopart.glade.h:9
msgid "_Resize"
msgstr "_Тағйири андоза"

#: ../iw/partition_ui_helpers_gui.py:409
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""

#: ../iw/progress_gui.py:37
#, fuzzy
msgid "Installing Packages"
msgstr "Чопгари %1 танзим шуда истодааст"

#: ../iw/raid_dialog_gui.py:312
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""

#: ../iw/raid_dialog_gui.py:326 ../iw/raid_dialog_gui.py:763
msgid "Make RAID Device"
msgstr "Эҷоди дастгоҳи RAID"

#: ../iw/raid_dialog_gui.py:329
#, fuzzy, python-format
msgid "Edit RAID Device: %s"
msgstr "Таҳрири дастгоҳи RAID"

#: ../iw/raid_dialog_gui.py:331
msgid "Edit RAID Device"
msgstr "Таҳрири дастгоҳи RAID"

#: ../iw/raid_dialog_gui.py:402
msgid "RAID _Device:"
msgstr "Дастгоҳи _RAID"

#: ../iw/raid_dialog_gui.py:420
msgid "RAID _Level:"
msgstr "Дараҷаи  _RAID:"

#: ../iw/raid_dialog_gui.py:462
#, fuzzy
msgid "_RAID Members:"
msgstr "Дараҷотҳои &ишорашуда"

#: ../iw/raid_dialog_gui.py:479
#, fuzzy
msgid "Number of _spares:"
msgstr "Рақами сабткунӣ"

#: ../iw/raid_dialog_gui.py:489
#, fuzzy
msgid "_Format partition?"
msgstr "Иловаи қисми диск"

#: ../iw/raid_dialog_gui.py:563
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""

#: ../iw/raid_dialog_gui.py:567 ../iw/raid_dialog_gui.py:573
#: ../iw/raid_dialog_gui.py:587 ../iw/raid_dialog_gui.py:600
msgid "Source Drive Error"
msgstr "Хатои манбаъи дастгоҳ"

#: ../iw/raid_dialog_gui.py:574
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""

#: ../iw/raid_dialog_gui.py:588
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""

#: ../iw/raid_dialog_gui.py:601
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""

#: ../iw/raid_dialog_gui.py:615 ../iw/raid_dialog_gui.py:621
#: ../iw/raid_dialog_gui.py:637
msgid "Target Drive Error"
msgstr "Хатои нишонад дастгоҳ"

#: ../iw/raid_dialog_gui.py:616
msgid "Please select the target drives for the clone operation."
msgstr ""

#: ../iw/raid_dialog_gui.py:622
#, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr ""

#: ../iw/raid_dialog_gui.py:638
#, python-format
msgid ""
"The target drive %s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""

#: ../iw/raid_dialog_gui.py:701
#, fuzzy
msgid "Please select a source drive."
msgstr "Фақат як алоқаро нишон диҳед."

#: ../iw/raid_dialog_gui.py:721
#, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""

#: ../iw/raid_dialog_gui.py:726
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""

#: ../iw/raid_dialog_gui.py:729
#, fuzzy
msgid "Final Warning"
msgstr "Параметри нодуруст"

#: ../iw/raid_dialog_gui.py:731
msgid "Clone Drives"
msgstr "Нусхаи диск"

#: ../iw/raid_dialog_gui.py:740
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr ""

#: ../iw/raid_dialog_gui.py:773
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required "
"to setup RAID arrays.  This tool uses a source drive which has been prepared "
"with the desired partitioning layout, and clones this layout onto other "
"similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on "
"that drive only, and can only contain unused software RAID partitions.  "
"Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""

#: ../iw/raid_dialog_gui.py:793
msgid "Source Drive:"
msgstr "Дастгоҳи манбаъ:"

#: ../iw/raid_dialog_gui.py:801
msgid "Target Drive(s):"
msgstr "Нишонаи дастгоҳ(s):"

#: ../iw/raid_dialog_gui.py:809
msgid "Drives"
msgstr "Дастгоҳҳо"

#: ../iw/task_gui.py:64
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""

#: ../iw/task_gui.py:162
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""

#: ../iw/task_gui.py:175
msgid "Edit Repository"
msgstr "Таҳрири феҳраст"

#: ../iw/task_gui.py:238
msgid "Invalid Proxy URL"
msgstr "Proxy URL нодуруст"

#: ../iw/task_gui.py:239
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr ""

#: ../iw/task_gui.py:250 ../iw/task_gui.py:388
msgid "Invalid Repository URL"
msgstr "Феҳраст URL нодуруст"

#: ../iw/task_gui.py:251 ../iw/task_gui.py:389
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr ""

#: ../iw/task_gui.py:273 ../iw/task_gui.py:398
msgid "No Media Found"
msgstr "Ягон дастгоҳ ёфт нашуд"

#: ../iw/task_gui.py:274 ../iw/task_gui.py:399
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""

#: ../iw/task_gui.py:293 ../iw/task_gui.py:415
msgid "Please enter an NFS server and path."
msgstr ""

#: ../iw/task_gui.py:325
msgid "Invalid Repository Name"
msgstr "Номи феҳрасти нодуруст"

#: ../iw/task_gui.py:326
#, fuzzy
msgid "You must provide a repository name."
msgstr "Шумо бояд номи нохолиро пешкаш намоед."

#: ../iw/task_gui.py:450 tmp/addrepo.glade.h:3
msgid "Add Repository"
msgstr "Иловаи феҳраст"

#: ../iw/task_gui.py:455
msgid "No Software Repos Enabled"
msgstr ""

#: ../iw/task_gui.py:456
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""

#: ../iw/timezone_gui.py:63 ../textw/timezone_text.py:96
msgid "Time Zone Selection"
msgstr "Интихоби минтақаи вақт"

#: ../iw/upgrade_bootloader_gui.py:36 ../textw/upgrade_bootloader_text.py:135
msgid "Upgrade Boot Loader Configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:122
msgid "_Update boot loader configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:123
msgid "This will update your current boot loader."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:127 ../textw/upgrade_bootloader_text.py:107
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:130 ../textw/upgrade_bootloader_text.py:111
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:137 ../textw/upgrade_bootloader_text.py:120
#, python-format
msgid ""
"The installer has detected the %s boot loader currently installed on %s."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:141
#, fuzzy
msgid "This is the recommended option."
msgstr "Сигнали номуайян"

#: ../iw/upgrade_bootloader_gui.py:146
msgid "_Create new boot loader configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:148
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:155
msgid "_Skip boot loader updating"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:156
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:167
#, fuzzy
msgid "What would you like to do?"
msgstr "Гузарвожае, ки истифода бурдан мехоҳед."

#: ../iw/upgrade_migratefs_gui.py:38 ../textw/upgrade_text.py:36
#, fuzzy
msgid "Migrate File Systems"
msgstr "Миқдори системаҳои қубурҳо"

#: ../iw/upgrade_migratefs_gui.py:70
#, python-format
msgid ""
"This release of %s supports an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: ../iw/upgrade_swap_gui.py:35
#, fuzzy
msgid "Upgrade Swap Partition"
msgstr "Матн: Иввази ранг"

#: ../iw/upgrade_swap_gui.py:94 ../textw/upgrade_text.py:101
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""

#: ../iw/upgrade_swap_gui.py:101
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""

#: ../iw/upgrade_swap_gui.py:113
#, fuzzy
msgid "I _want to create a swap file"
msgstr "Файли муваққатиро офарида намешавад."

#: ../iw/upgrade_swap_gui.py:122
msgid "Select the _partition to put the swap file on:"
msgstr ""

#: ../iw/upgrade_swap_gui.py:140 ../textw/upgrade_text.py:119
msgid "Partition"
msgstr "Қисми диск"

#: ../iw/upgrade_swap_gui.py:140
msgid "Free Space (MB)"
msgstr "Ҷои холӣ (МБ)"

#: ../iw/upgrade_swap_gui.py:158
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""

#: ../iw/upgrade_swap_gui.py:173
#, fuzzy
msgid "Swap file _size (MB):"
msgstr "&Андозаи тасвири файлро нишон диҳед"

#: ../iw/upgrade_swap_gui.py:183
msgid "I _don't want to create a swap file"
msgstr ""

#: ../iw/upgrade_swap_gui.py:193
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""

#: ../iw/upgrade_swap_gui.py:201 ../textw/upgrade_text.py:188
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr ""

#: ../iw/upgrade_swap_gui.py:208 ../textw/upgrade_text.py:183
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""

#: ../iw/welcome_gui.py:56 ../textw/welcome_text.py:36
msgid "Network Install Required"
msgstr ""

#: ../iw/welcome_gui.py:57 ../textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""

#: ../iw/welcome_gui.py:67
#, fuzzy
msgid "E_xit Installer"
msgstr "Хуруҷи сабткунанда"

#: ../iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr ""

#: ../iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr ""

#: ../iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""

#: ../iw/zipl_gui.py:90 ../textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr "Параметрҳои Kernel"

#: ../iw/zipl_gui.py:93 ../iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "Параметриҳои Chandev"

#: ../textw/complete_text.py:32
msgid "Reboot"
msgstr "Бозхудборшавӣ"

#: ../textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "Тугмаи <Enter> барои хуруҷ"

#: ../textw/complete_text.py:62
msgid "Complete"
msgstr "Ҳама"

#: ../textw/constants_text.py:52 ../storage/__init__.py:135
#: ../storage/devicetree.py:78 ../loader/cdinstall.c:452
#: ../loader/driverdisk.c:244 ../loader/driverdisk.c:275
#: ../loader/driverdisk.c:359 ../loader/driverselect.c:81
#: ../loader/driverselect.c:179 ../loader/driverselect.c:205
#: ../loader/hdinstall.c:231 ../loader/hdinstall.c:289 ../loader/kbd.c:138
#: ../loader/loader.c:482 ../loader/loader.c:502 ../loader/loader.c:1378
#: ../loader/loader.c:1409 ../loader/net.c:564 ../loader/net.c:937
#: ../loader/net.c:1808 ../loader/net.c:1830 ../loader/nfsinstall.c:91
#: ../loader/urls.c:292
msgid "Back"
msgstr "Ақиб"

#: ../textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "Интихоби клавиатура"

#: ../textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr ""

#: ../textw/netconfig_text.py:41
#, fuzzy, python-format
msgid "A value is required for the field %s"
msgstr "Маънои\"%1\" барои драйвер аввалиндараҷа нест."

#: ../textw/netconfig_text.py:86 tmp/netconfig.glade.h:9
msgid "Enable network interface"
msgstr "Танзимотҳои Интернетро фаъол созед"

#: ../textw/netconfig_text.py:89 tmp/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: ../textw/netconfig_text.py:127
msgid "Use dynamic IP configuration (DHCP)"
msgstr "Танзимоти динамикии IP (DHCP) истифода баред"

#: ../textw/netconfig_text.py:130 ../loader/net.c:542
msgid "Enable IPv4 support"
msgstr "Пуштибонии IPv4 фаъол созед"

#: ../textw/netconfig_text.py:137
msgid "IPv4 Address:"
msgstr "Суроғаи IPv4:"

#: ../textw/netconfig_text.py:157 ../loader/net.c:896
msgid "Gateway:"
msgstr "Суроғаи хидматгоҳ:"

#: ../textw/netconfig_text.py:160
msgid "Nameserver:"
msgstr "Номи хидматгоҳ:"

#: ../textw/netconfig_text.py:190
msgid "Missing Device"
msgstr "Дастгоҳ мавҷуд нест"

#: ../textw/netconfig_text.py:191
#, fuzzy
msgid "You must select a network device"
msgstr "Шумо бояд якчанд ячейка интихоб кунед."

#: ../textw/netconfig_text.py:242
#, fuzzy
msgid "IPv4 Network Mask "
msgstr "Ниқоби бехатарии феҳрист:"

#: ../textw/netconfig_text.py:265
#, fuzzy
msgid "Configuring Network Interfaces"
msgstr "Ба ҳам пайвастани танҳо интерфейсҳо"

#: ../textw/netconfig_text.py:265
#, fuzzy
msgid "Waiting for NetworkManager..."
msgstr "интизории ҷавоб"

#: ../textw/netconfig_text.py:271
#, fuzzy
msgid "Error configuring network device"
msgstr "Хатои дохилӣ: Дастгоҳ танзим нашудааст."

#: ../textw/netconfig_text.py:271
#, fuzzy, python-format
msgid "Error configuring network device %s"
msgstr "Хатои дохилӣ: Дастгоҳ танзим нашудааст."

#: ../textw/partition_text.py:58
msgid "Partitioning Type"
msgstr "Намуди қисмҳои диск"

#: ../textw/partition_text.py:60 tmp/autopart.glade.h:2
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target. You can also choose to create your own custom "
"layout."
msgstr ""

#: ../textw/partition_text.py:76
msgid "Which drive(s) do you want to use for this installation?"
msgstr ""

#: ../textw/partition_text.py:91
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr ""

#: ../textw/partition_text.py:165 tmp/adddrive.glade.h:3
msgid "Advanced Storage Options"
msgstr "Интихобҳои иловагии захирагоҳ"

#: ../textw/partition_text.py:166 tmp/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr ""

#: ../textw/partition_text.py:189
msgid "Add FCP Device"
msgstr "Иловаи дастгоҳи FCP"

#: ../textw/partition_text.py:190 tmp/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""

#: ../textw/partition_text.py:213 tmp/iscsi-config.glade.h:7
msgid "Configure iSCSI Parameters"
msgstr "Танзимоти параметрҳои iSCSI"

#: ../textw/partition_text.py:214 tmp/iscsi-config.glade.h:8
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: ../textw/partition_text.py:215
msgid "Target IP Address"
msgstr "Суроғаи нишонаи IP"

#: ../textw/partition_text.py:216
msgid "iSCSI Initiator Name"
msgstr "Номи iSCSI Initiator"

#: ../textw/partition_text.py:217
#, fuzzy
msgid "CHAP username"
msgstr "&Сатҳи номи корванд:"

#: ../textw/partition_text.py:218
#, fuzzy
msgid "CHAP password"
msgstr "Бе парол"

#: ../textw/partition_text.py:219
#, fuzzy
msgid "Reverse CHAP username"
msgstr "Имя + пароль (классика)"

#: ../textw/partition_text.py:220
#, fuzzy
msgid "Reverse CHAP password"
msgstr "Истифодаи пароли GRUB"

#: ../textw/progress_text.py:46
msgid "Package Installation"
msgstr "Сабткунии қуттиҳо"

#: ../textw/timezone_text.py:75
msgid "In which time zone are you located?"
msgstr ""

#: ../textw/timezone_text.py:93
#, fuzzy
msgid "System clock uses UTC"
msgstr "Барпои муҳити соат:  UTC: %1  JD: %2"

#: ../textw/upgrade_bootloader_text.py:115
#: ../textw/upgrade_bootloader_text.py:124
msgid "Update boot loader configuration"
msgstr ""

#: ../textw/upgrade_bootloader_text.py:127
msgid "Skip boot loader updating"
msgstr ""

#: ../textw/upgrade_bootloader_text.py:129
msgid "Create new boot loader configuration"
msgstr ""

#: ../textw/upgrade_text.py:38
#, python-format
msgid ""
"This release of %s supports the an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: ../textw/upgrade_text.py:119
msgid "Free Space"
msgstr "Ҷои холӣ"

#: ../textw/upgrade_text.py:137
msgid "RAM detected (MB):"
msgstr "RAM муайяншуда (MB):"

#: ../textw/upgrade_text.py:140
#, fuzzy
msgid "Suggested size (MB):"
msgstr "Ҷои холӣ (МБ)"

#: ../textw/upgrade_text.py:143
#, fuzzy
msgid "Swap file size (MB):"
msgstr "&Андозаи тасвири файлро нишон диҳед"

#: ../textw/upgrade_text.py:151
msgid "Add Swap"
msgstr "Иловаи Swap"

#: ../textw/upgrade_text.py:176
msgid "The value you entered is not a valid number."
msgstr ""

#: ../textw/upgrade_text.py:209
msgid "Reinstall System"
msgstr "Сабткунии системаи нав"

#: ../textw/upgrade_text.py:219
msgid "System to Upgrade"
msgstr "Навсозии система"

#: ../textw/upgrade_text.py:220
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""

#: ../textw/userauth_text.py:30
msgid "Root Password"
msgstr "Пароли Root"

#: ../textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""

#: ../textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr ""

#: ../textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: ../textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"Марҳамат намоед ба %s!\n"
"\n"

#: ../textw/zipl_text.py:35
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""

#: ../textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr "Танзимоти z/IPL"

#: ../textw/zipl_text.py:75 ../textw/zipl_text.py:79
#, fuzzy
msgid "Chandev line "
msgstr "Сатри фармон:"

#: ../installclasses/fedora.py:39
msgid "_Fedora"
msgstr "_Fedora"

#: ../installclasses/fedora.py:40 ../installclasses/rhel.py:45
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""

#: ../installclasses/fedora.py:49
msgid "Office and Productivity"
msgstr "Идора ва сермаҳсулӣ"

#: ../installclasses/fedora.py:50 ../installclasses/rhel.py:57
#: ../installclasses/rhel.py:62
msgid "Software Development"
msgstr "Барноманависии нармафзор"

#: ../installclasses/fedora.py:51
#, fuzzy
msgid "Web Server"
msgstr "Хидматгоҳи Веб"

#: ../installclasses/rhel.py:44
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: ../installclasses/rhel.py:54
msgid "Office"
msgstr "Офис"

#: ../installclasses/rhel.py:55
msgid "Multimedia"
msgstr "Мултимедиа"

#: ../installclasses/rhel.py:61
msgid "Web server"
msgstr "Хидматгоҳи Интернет"

#: ../installclasses/rhel.py:66
msgid "Virtualization"
msgstr "Барномаҳои виртуалӣ"

#: ../installclasses/rhel.py:67
msgid "Clustering"
msgstr "Кластерҳо"

#: ../installclasses/rhel.py:68
msgid "Storage Clustering"
msgstr "Захирагоҳи кластерҳо"

#: ../installclasses/rhel.py:72
msgid "Installation Number"
msgstr "Рақами сабткунӣ"

#: ../installclasses/rhel.py:73
msgid ""
"To install the full set of supported packages included in your subscription, "
"please enter your Installation Number"
msgstr ""

#: ../installclasses/rhel.py:76
msgid ""
"If you're unable to locate the Installation Number, consult http://www."
"redhat.com/apps/support/in.html.\n"
"\n"
"If you skip:\n"
"* You may not get access to the full set of packages included in your "
"subscription.\n"
"* It may result in an unsupported/uncertified installation of Red Hat "
"Enterprise Linux.\n"
"* You will not get software and security updates for packages not included "
"in your subscription."
msgstr ""

#: ../storage/__init__.py:81
#, fuzzy
msgid "Unknown Device"
msgstr "Дастгоҳи Номаълум"

#: ../storage/__init__.py:82
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""

#: ../storage/__init__.py:95
msgid "Installation cannot continue."
msgstr "Сабткунӣ давом дода намешавад"

#: ../storage/__init__.py:96
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""

#: ../storage/__init__.py:127
msgid "Encrypt device?"
msgstr "Дастгрҳро рамзгузорӣ мекунед"

#: ../storage/__init__.py:128
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""

#: ../storage/__init__.py:146
#, fuzzy
msgid "Writing storage configuration to disk"
msgstr "Ҳангоми навишти PTY хато пайдо шуд"

#: ../storage/__init__.py:147
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""

#: ../storage/__init__.py:152
#, fuzzy
msgid "Go _back"
msgstr "Гузаштан ба қафо"

#: ../storage/__init__.py:153
#, fuzzy
msgid "_Write changes to disk"
msgstr "Тағиротҳоро дар санадоти %1 нигоҳ дорам?"

#: ../storage/__init__.py:252
#, fuzzy
msgid "Finding Devices"
msgstr "Дастгоҳ мавҷуд нест"

#: ../storage/__init__.py:253
msgid "Finding storage devices..."
msgstr ""

#: ../storage/__init__.py:269
msgid "Filesystem error detected, cannot continue."
msgstr ""

#: ../storage/__init__.py:480
msgid "This partition is holding the data for the hard drive install."
msgstr ""

#: ../storage/__init__.py:485
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""

#: ../storage/__init__.py:491
#, python-format
msgid "This device is part of the RAID device %s."
msgstr ""

#: ../storage/__init__.py:494
msgid "This device is part of a RAID device."
msgstr ""

#: ../storage/__init__.py:499
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr ""

#: ../storage/__init__.py:502
msgid "This device is part of a LVM volume group."
msgstr ""

#: ../storage/__init__.py:511
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""

#: ../storage/__init__.py:751
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: ../storage/__init__.py:756
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: ../storage/__init__.py:762
#, python-format
msgid ""
"Your / partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""

#: ../storage/__init__.py:770
#, python-format
msgid ""
"Your %s partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""

#: ../storage/__init__.py:797
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: ../storage/__init__.py:800
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: ../storage/__init__.py:804
#, fuzzy
msgid "You have not created a boot partition."
msgstr "Шумо наметавонед папкаи системавиро ҳузф кунед."

#: ../storage/__init__.py:808
msgid "Bootable partitions can only be on RAID1 devices."
msgstr ""

#: ../storage/__init__.py:813
msgid "Bootable partitions cannot be on a logical volume."
msgstr ""

#: ../storage/__init__.py:818
msgid "Bootable partitions cannot be on a RAID device."
msgstr ""

#: ../storage/__init__.py:823 ../storage/__init__.py:828
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr ""

#: ../storage/__init__.py:833
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""

#: ../storage/__init__.py:837
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: ../storage/__init__.py:979 ../storage/__init__.py:988
#, fuzzy
msgid "Dirty File Systems"
msgstr "Миқдори системаҳои қубурҳо"

#: ../storage/__init__.py:980
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""

#: ../storage/__init__.py:989
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: ../storage/__init__.py:1406
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: ../storage/__init__.py:1414
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""

#: ../storage/__init__.py:1428
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""

#: ../storage/__init__.py:1434
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""

#: ../storage/__init__.py:1475 ../storage/__init__.py:1485
msgid "Invalid mount point"
msgstr "Нуқтаи васли нодуруст"

#: ../storage/__init__.py:1476
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: ../storage/__init__.py:1486
#, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: ../storage/__init__.py:1499 ../storage/__init__.py:1520
#, fuzzy
msgid "Unable to mount filesystem"
msgstr "Не удаётся выполнить /sbin/mount."

#: ../storage/__init__.py:1500
#, python-format
msgid ""
"An error occurred mounting device %s as %s.  You may continue installation, "
"but there may be problems."
msgstr ""

#: ../storage/__init__.py:1521
#, python-format
msgid ""
"An error occurred mounting device %s as %s: %s. This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: ../storage/devices.py:1059
#, fuzzy
msgid "boot flag not available for this partition"
msgstr "Шумо наметавонед папкаи системавиро ҳузф кунед."

#: ../storage/devicetree.py:87
msgid "Confirm"
msgstr "Тасдиқ"

#: ../storage/devicetree.py:88
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""

#: ../storage/devicetree.py:126
#, python-format
msgid ""
"Error processing drive %s.\n"
"Maybe it needs to be reinitialized.YOU WILL LOSE ALL DATA ON THIS DRIVE!"
msgstr ""

#: ../storage/devicetree.py:151
#, python-format
msgid ""
"Error processing LVM.\n"
"It seems that there is inconsistent LVM data. (%s) make(s) up %s. You can "
"reinitialize all related PVs, which will erase all LVM metadata. Or ignore, "
"which will preserve contents."
msgstr ""

#: ../storage/devicetree.py:159
#, fuzzy
msgid "_Ignore drive(s)"
msgstr "_Радкунии дастгоҳ"

#: ../storage/devicetree.py:160
#, fuzzy
msgid "_Re-initialize drive(s)"
msgstr "Танзимоти дастгоҳи лента ҷамъшаванда"

#: ../storage/iscsi.py:98 ../storage/iscsi.py:99
#, fuzzy
msgid "Scanning iSCSI nodes"
msgstr "Боркунии драйвери SCSI"

#: ../storage/iscsi.py:172 ../storage/iscsi.py:173
#, fuzzy
msgid "Initializing iSCSI initiator"
msgstr "Дар мавриди қарор гирифтани системаи чопгар..."

#: ../storage/iscsi.py:208
#, fuzzy
msgid "iSCSI not available"
msgstr "Ёрӣ дастнорас аст"

#: ../storage/iscsi.py:210
#, fuzzy
msgid "No initiator name set"
msgstr "Номи iSCSI Initiator"

#: ../storage/iscsi.py:224
msgid "No iSCSI nodes discovered"
msgstr ""

#: ../storage/iscsi.py:227 ../storage/iscsi.py:228
msgid "Logging in to iSCSI nodes"
msgstr ""

#: ../storage/iscsi.py:250
msgid "No new iSCSI nodes discovered"
msgstr ""

#: ../storage/iscsi.py:253
msgid "Could not log in to any of the discovered nodes"
msgstr ""

#: ../storage/partitioning.py:175
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""

#: ../storage/partitioning.py:192
msgid "Warnings During Automatic Partitioning"
msgstr ""

#: ../storage/partitioning.py:194
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""

#: ../storage/partitioning.py:206 ../storage/partitioning.py:230
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""

#: ../storage/partitioning.py:208
#, fuzzy, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr "Амалиёти дархостшуда иҷро карда намешавад"

#: ../storage/partitioning.py:232
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: ../storage/partitioning.py:234
#, fuzzy
msgid "Automatic Partitioning Errors"
msgstr "Хатогӣ қабул шуд:"

#: ../storage/partitioning.py:235
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %s"
msgstr ""

#: ../storage/partitioning.py:246
#, fuzzy
msgid "Unrecoverable Error"
msgstr "Хатои Kickstart"

#: ../storage/partitioning.py:247
msgid "Your system will now be rebooted."
msgstr ""

#: ../storage/zfcp.py:52
msgid "You have not specified a device number or the number is invalid"
msgstr ""

#: ../storage/zfcp.py:54
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""

#: ../storage/zfcp.py:56
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr ""

#: ../storage/devicelibs/lvm.py:326
#, python-format
msgid "vginfo failed for %s"
msgstr ""

#: ../storage/devicelibs/lvm.py:354
#, python-format
msgid "lvs failed for %s"
msgstr ""

#: ../storage/formats/fs.py:64
msgid "attr dict must include a type"
msgstr ""

#: ../storage/formats/fs.py:111
msgid "filesystem configuration missing a type"
msgstr ""

#: ../storage/formats/fs.py:270
#, fuzzy
msgid "Formatting"
msgstr "Шаклбандӣ"

#: ../storage/formats/fs.py:271
#, fuzzy, python-format
msgid "Creating filesystem on %s..."
msgstr "Ба ҳамон системаи файлӣ қарор гирифтан"

#: ../storage/formats/fs.py:367
#, fuzzy
msgid "Resizing"
msgstr "&Ҳаракатдиҳӣ/бозандозагирӣ"

#: ../storage/formats/fs.py:368
#, fuzzy, python-format
msgid "Resizing filesystem on %s..."
msgstr "Ба ҳамон системаи файлӣ қарор гирифтан"

#: ../storage/formats/fs.py:409
msgid "Checking"
msgstr "Озмоиш мекунад"

#: ../storage/formats/fs.py:410
#, fuzzy, python-format
msgid "Checking filesystem on %s..."
msgstr "Ба ҳамон системаи файлӣ қарор гирифтан"

#: ../loader/cdinstall.c:185 ../loader/cdinstall.c:206
#: ../loader/mediacheck.c:60
msgid "Media Check"
msgstr "Санҷиши медиа"

#: ../loader/cdinstall.c:185 ../loader/cdinstall.c:188
#: ../loader/cdinstall.c:206 ../loader/cdinstall.c:214 ../loader/method.c:326
msgid "Test"
msgstr "Санҷиш"

#: ../loader/cdinstall.c:185 ../loader/cdinstall.c:189
msgid "Eject Disc"
msgstr "Баровардани диск"

#: ../loader/cdinstall.c:186
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""

#: ../loader/cdinstall.c:207
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""

#: ../loader/cdinstall.c:229
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""

#: ../loader/cdinstall.c:248
msgid "Disc Found"
msgstr "Диск ёфт шуд"

#: ../loader/cdinstall.c:249
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""

#: ../loader/cdinstall.c:339
msgid "Scanning"
msgstr "Сканеркунӣ"

#: ../loader/cdinstall.c:339 ../loader/cdinstall.c:341
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr ""

#: ../loader/cdinstall.c:442
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""

#: ../loader/cdinstall.c:451
msgid "Disc Not Found"
msgstr "Диск ёфт нашуд"

#: ../loader/cdinstall.c:518
msgid "Cannot find kickstart file on CDROM."
msgstr ""

#: ../loader/copy.c:51 ../loader/method.c:280
#, fuzzy, c-format
msgid "Failed to read directory %s: %m"
msgstr "Сана хонда натавонист."

#: ../loader/driverdisk.c:139
msgid "Loading"
msgstr "Поркунӣ"

#: ../loader/driverdisk.c:139
msgid "Reading driver disk..."
msgstr "Озмоиши диски драйвер..."

#: ../loader/driverdisk.c:238 ../loader/driverdisk.c:270
msgid "Driver Disk Source"
msgstr "Манбаъи диски драйвер"

#: ../loader/driverdisk.c:239
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""

#: ../loader/driverdisk.c:271
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""

#: ../loader/driverdisk.c:306
#, fuzzy
msgid "Failed to mount partition."
msgstr "ХАТО вакти нобуд сохтани %1 аз %2: \"%3\"\n"

#: ../loader/driverdisk.c:311
#, fuzzy
msgid "Select driver disk image"
msgstr "Интихоби файли тасвири мизи корӣ"

#: ../loader/driverdisk.c:312
msgid "Select the file which is your driver disk image."
msgstr ""

#: ../loader/driverdisk.c:341
msgid "Failed to load driver disk from file."
msgstr ""

#: ../loader/driverdisk.c:353
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: ../loader/driverdisk.c:359
msgid "Insert Driver Disk"
msgstr "Вориди диски драйвер"

#: ../loader/driverdisk.c:372
msgid "Failed to mount driver disk."
msgstr ""

#: ../loader/driverdisk.c:380
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: ../loader/driverdisk.c:443
#, fuzzy
msgid "Manually choose"
msgstr "Забонро интихоб кунед"

#: ../loader/driverdisk.c:444
msgid "Load another disk"
msgstr "Диски дигарро истифода баред"

#: ../loader/driverdisk.c:445
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""

#: ../loader/driverdisk.c:483
msgid "Driver disk"
msgstr "Диски драйвер"

#: ../loader/driverdisk.c:484
msgid "Do you have a driver disk?"
msgstr "Шумо драйвери диск доред?"

#: ../loader/driverdisk.c:493
#, fuzzy
msgid "More Driver Disks?"
msgstr "Ягон драйвер ёфт нашуд"

#: ../loader/driverdisk.c:494
msgid "Do you wish to load any more driver disks?"
msgstr ""

#: ../loader/driverdisk.c:539 ../loader/driverdisk.c:576
#: ../loader/hdinstall.c:384 ../loader/kickstart.c:132
#: ../loader/kickstart.c:142 ../loader/kickstart.c:185
#: ../loader/kickstart.c:553 ../loader/modules.c:335 ../loader/modules.c:347
#: ../loader/net.c:1541 ../loader/net.c:1564 ../loader/nfsinstall.c:335
#: ../loader/urlinstall.c:418 ../loader/urlinstall.c:427
#: ../loader/urlinstall.c:436
msgid "Kickstart Error"
msgstr "Хатои Kickstart"

#: ../loader/driverdisk.c:540
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr ""

#: ../loader/driverdisk.c:577
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""

#: ../loader/driverselect.c:67
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: ../loader/driverselect.c:91
msgid "Enter Module Parameters"
msgstr "Вориди параметрҳои модул"

#: ../loader/driverselect.c:178
msgid "No drivers found"
msgstr "Ягон драйвер ёфт нашуд"

#: ../loader/driverselect.c:178
msgid "Load driver disk"
msgstr "Интихоби диски драйвер"

#: ../loader/driverselect.c:179
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""

#: ../loader/driverselect.c:197
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""

#: ../loader/driverselect.c:206
#, fuzzy
msgid "Specify optional module arguments"
msgstr "Номи мидулро ишора кунед."

#: ../loader/driverselect.c:234
#, fuzzy
msgid "Select Device Driver to Load"
msgstr "<qt>Қобили пурбор кардани драйвери лозимӣ нестам:<p>%1</p></qt>"

#: ../loader/hdinstall.c:144
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""

#: ../loader/hdinstall.c:232
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""

#: ../loader/hdinstall.c:246
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""

#: ../loader/hdinstall.c:273
#, fuzzy
msgid "Directory holding image:"
msgstr "Тасвири силсиларо эҷод кунед"

#: ../loader/hdinstall.c:301
msgid "Select Partition"
msgstr "Интихоби қисми диск"

#: ../loader/hdinstall.c:353
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr ""

#: ../loader/hdinstall.c:385
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr ""

#: ../loader/hdinstall.c:455 ../loader/hdinstall.c:511
msgid "Cannot find kickstart file on hard drive."
msgstr ""

#: ../loader/hdinstall.c:498
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr ""

#: ../loader/kbd.c:136
msgid "Keyboard Type"
msgstr "Намуди клавиатура"

#: ../loader/kbd.c:137
#, fuzzy
msgid "What type of keyboard do you have?"
msgstr "Шумо чӣ мехоҳед кард?"

#: ../loader/kickstart.c:133
#, fuzzy, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "Хатогӣ ҳангоми кушодани файл"

#: ../loader/kickstart.c:143
#, fuzzy, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "Ҳангоми хондани файл хато пайдо шуд '%1'"

#: ../loader/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr ""

#: ../loader/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr ""

#: ../loader/kickstart.c:380
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: ../loader/kickstart.c:389
#, fuzzy
msgid "Error downloading kickstart file"
msgstr "Хатогӣ: Якҷоянашавандаи шакли файлҳо"

#: ../loader/kickstart.c:554
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr ""

#: ../loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "Марҳамат намоед ба %s барои %s - Ҳолати бехатарӣ"

#: ../loader/lang.c:65 ../loader/loader.c:226
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""

#: ../loader/lang.c:375
msgid "Choose a Language"
msgstr "Забонро интихоб кунед"

#: ../loader/loader.c:127
msgid "Local CD/DVD"
msgstr "Дискҳои маҳаллии CD/DVD"

#: ../loader/loader.c:128
msgid "Hard drive"
msgstr "Диски компютерӣ"

#: ../loader/loader.c:129
msgid "NFS directory"
msgstr "Феҳрасти NFS"

#: ../loader/loader.c:433 ../loader/loader.c:477
msgid "Update Disk Source"
msgstr "Навсозии манбаъи диск"

#: ../loader/loader.c:434
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""

#: ../loader/loader.c:478
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""

#: ../loader/loader.c:496
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""

#: ../loader/loader.c:502
msgid "Updates Disk"
msgstr "Дискро аз нав месозад"

#: ../loader/loader.c:520
msgid "Failed to mount updates disk"
msgstr ""

#: ../loader/loader.c:525
msgid "Updates"
msgstr "Навсозиҳо"

#: ../loader/loader.c:525
msgid "Reading anaconda updates..."
msgstr "Навсозии anaconda хонда шудааст..."

#: ../loader/loader.c:561
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""

#: ../loader/loader.c:570
msgid "Error downloading updates image"
msgstr "Навсозии бойгонии диск файрон аст"

#: ../loader/loader.c:1196
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr ""

#: ../loader/loader.c:1253
msgid "Media Detected"
msgstr "Дастгоҳ муайян карда шуд"

#: ../loader/loader.c:1254
msgid "Local installation media detected..."
msgstr ""

#: ../loader/loader.c:1370
msgid "Rescue Method"
msgstr "Тарзи барқарорсозӣ"

#: ../loader/loader.c:1371
msgid "Installation Method"
msgstr "Тарзи сабткунӣ"

#: ../loader/loader.c:1373
msgid "What type of media contains the rescue image?"
msgstr ""

#: ../loader/loader.c:1375
msgid "What type of media contains the installation image?"
msgstr ""

#: ../loader/loader.c:1408
msgid "No driver found"
msgstr "Ягон драйвер ёфт нашуд"

#: ../loader/loader.c:1408
msgid "Select driver"
msgstr "Интихоби драйвер"

#: ../loader/loader.c:1409
msgid "Use a driver disk"
msgstr "Интихоби диски драйвер"

#: ../loader/loader.c:1410
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""

#: ../loader/loader.c:1630
msgid "The following devices have been found on your system."
msgstr ""

#: ../loader/loader.c:1632
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""

#: ../loader/loader.c:1636
msgid "Devices"
msgstr "Дастгоҳҳо"

#: ../loader/loader.c:1637
msgid "Done"
msgstr "Шуд"

#: ../loader/loader.c:1638
msgid "Add Device"
msgstr "Иловаи дастгоҳ"

#: ../loader/loader.c:1850
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr ""

#: ../loader/loader.c:2224
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait...\n"
msgstr ""

#: ../loader/loader.c:2226
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait...\n"
msgstr ""

#: ../loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr "Бойгонии диски %s барои сабт ёфт карда нашуд"

#: ../loader/mediacheck.c:52
#, c-format
msgid "Checking \"%s\"..."
msgstr "Озмоиши \"%s\"..."

#: ../loader/mediacheck.c:54
#, c-format
msgid "Checking media now..."
msgstr "Озмоиши медиа..."

#: ../loader/mediacheck.c:86
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""

#: ../loader/mediacheck.c:93
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: ../loader/mediacheck.c:101
msgid "Success"
msgstr "Мувафақият"

#: ../loader/mediacheck.c:102
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""

#: ../loader/method.c:323
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""

#: ../loader/method.c:326
msgid "Checksum Test"
msgstr "Санҷиши Checksum"

#: ../loader/modules.c:336
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr ""

#: ../loader/modules.c:348
msgid "A module name must be specified for the kickstart device command."
msgstr ""

#: ../loader/net.c:111
msgid "Invalid Prefix"
msgstr "Префикси нодуруст"

#: ../loader/net.c:112
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: ../loader/net.c:454 ../loader/net.c:510
msgid "Network Error"
msgstr "Хатои пайвастшавии Интернет"

#: ../loader/net.c:455 ../loader/net.c:511
msgid "There was an error configuring your network interface."
msgstr ""

#: ../loader/net.c:556
msgid "Enable IPv6 support"
msgstr "Пуштибонии IPv6 фаъол созед"

#: ../loader/net.c:596
msgid "Configure TCP/IP"
msgstr "Танзимоти TCP/IP"

#: ../loader/net.c:655
msgid "Missing Protocol"
msgstr "Протокол мавҷуд нест"

#: ../loader/net.c:656
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr ""

#: ../loader/net.c:663
msgid "IPv4 Needed for NFS"
msgstr "IPv4 барои NFS лозим аст"

#: ../loader/net.c:664
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: ../loader/net.c:763
msgid "IPv4 address:"
msgstr "Суроғаи IPv4:"

#: ../loader/net.c:775 ../loader/net.c:842 tmp/netconfig.glade.h:1
msgid "/"
msgstr "/"

#: ../loader/net.c:830
msgid "IPv6 address:"
msgstr "Суроғаи IPv6:"

#: ../loader/net.c:904
msgid "Name Server:"
msgstr "Номи хидматгоҳ:"

#: ../loader/net.c:943
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: ../loader/net.c:963
msgid "Manual TCP/IP Configuration"
msgstr "Танзимоти дастӣ барои TCP/IP"

#: ../loader/net.c:1086 ../loader/net.c:1093
msgid "Missing Information"
msgstr "Иттилоот мавҷуд нест"

#: ../loader/net.c:1087
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""

#: ../loader/net.c:1094
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""

#: ../loader/net.c:1542
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr ""

#: ../loader/net.c:1565
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr ""

#: ../loader/net.c:1637
msgid "Seconds:"
msgstr "Дақиқа:"

#: ../loader/net.c:1803
msgid "Networking Device"
msgstr "Дастгоҳи Интернетӣ"

#: ../loader/net.c:1804
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""

#: ../loader/net.c:1808
msgid "Identify"
msgstr "Муайян кунед"

#: ../loader/net.c:1817
msgid "You can identify the physical port for"
msgstr ""

#: ../loader/net.c:1819
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""

#: ../loader/net.c:1829
msgid "Identify NIC"
msgstr "Муайянкунии NIC"

#: ../loader/net.c:1842
msgid "Invalid Duration"
msgstr "Давомнокии нодуруст"

#: ../loader/net.c:1843
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr ""

#: ../loader/net.c:1855
#, c-format
msgid "Flashing %s port lights for %d seconds..."
msgstr ""

#: ../loader/net.c:2015 ../loader/net.c:2019
#, c-format
msgid "Waiting for NetworkManager to configure %s...\n"
msgstr "Дар ҳоли интизории Мудири Шабака барои танзимоти  %s...\n"

#: ../loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "Номи хидматгоҳи NFS:"

#: ../loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "Феҳрасти %s:"

#: ../loader/nfsinstall.c:83
#, c-format
msgid "Please enter the server name and path to your %s installation image."
msgstr ""

#: ../loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "Танзими NFS"

#: ../loader/nfsinstall.c:249
msgid "That directory could not be mounted from the server."
msgstr ""

#: ../loader/nfsinstall.c:260
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr ""

#: ../loader/nfsinstall.c:336
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr ""

#: ../loader/telnetd.c:89 ../loader/telnetd.c:128
msgid "Telnet"
msgstr "Telnet"

#: ../loader/telnetd.c:89
msgid "Waiting for telnet connection..."
msgstr "Дар ҳоли интизории пайвастшавии telnet..."

#: ../loader/telnetd.c:128
msgid "Running anaconda via telnet..."
msgstr ""

#: ../loader/urlinstall.c:84
#, c-format
msgid "Unable to retrieve %s://%s%s."
msgstr "Қабул кардани %s://%s/%s имконпазир нест."

#: ../loader/urlinstall.c:176
msgid "Unable to retrieve the install image."
msgstr "Бойгонии дискро қабул карда натавонист."

#: ../loader/urlinstall.c:419
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr ""

#: ../loader/urlinstall.c:428
msgid "Must supply a --url argument to Url kickstart method."
msgstr ""

#: ../loader/urlinstall.c:437
#, c-format
msgid "Unknown Url method %s"
msgstr "Услуби Url %s номуайян"

#: ../loader/urls.c:235 ../loader/urls.c:237 ../loader/urls.c:240
msgid "Retrieving"
msgstr "Қабул мекунад"

#: ../loader/urls.c:295
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""

#: ../loader/urls.c:321
msgid "URL Setup"
msgstr "Танзимоти URL"

#: ../loader/urls.c:329
msgid "You must enter a URL."
msgstr "Шумо бояд URL-ро ворид кунед."

#: ../loader/urls.c:335
msgid "URL must be either an ftp or http URL"
msgstr ""

#: ../loader/urls.c:346
msgid "Unknown Host"
msgstr "Алоқаи номуайян"

#: ../loader/urls.c:347
#, c-format
msgid "%s is not a valid hostname."
msgstr "%s номи шабакаи нодуруст мебошад."

#: ../loader/windows.c:65
msgid "Loading SCSI driver"
msgstr "Боркунии драйвери SCSI"

#: ../loader/windows.c:66
#, c-format
msgid "Loading %s driver..."
msgstr "Боркунии драйвери %s..."

#: tmp/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""

#: tmp/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr "&Азқайдгирӣ"

#: tmp/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr ""

#: tmp/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr "_Брномаҳои интихобӣ"

#: tmp/GroupSelector.glade.h:5
msgid "_Select"
msgstr "_Интихоб"

#: tmp/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr "_Ҳамаи барномаҳоро интихоб кунед"

#: tmp/GroupSelector.glade.h:7
msgid "dialog1"
msgstr "диалоги1"

#: tmp/account.glade.h:1
msgid "Confirm:"
msgstr "Тасдиқ кунед:"

#: tmp/account.glade.h:2
msgid "Root Password:"
msgstr "Пароли Root:"

#: tmp/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: tmp/adddrive.glade.h:1
msgid "Add _ZFCP LUN"
msgstr "Иловаи _ZFCP LUN"

#: tmp/adddrive.glade.h:2
msgid "Add _iSCSI target"
msgstr "Иловаи _нишонаи iSCSI"

#: tmp/adddrive.glade.h:4
msgid "Disable _dmraid device"
msgstr "Хомӯшсозии _дастгоҳи dmraid"

#: tmp/adddrive.glade.h:6
msgid "_Add drive"
msgstr "_Иловаи дастгоҳ"

#: tmp/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr "<b>Номи _феҳраст:</b>"

#: tmp/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr "<b>Навъи _феҳраст:</b>"

#: tmp/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr "Танзимоти _proxy"

#: tmp/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""

#: tmp/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr ""

#: tmp/addrepo.glade.h:10
#, fuzzy
msgid "Proxy U_RL (host:port)"
msgstr "Сервер: %1 порт: %2 аллакай вуҷуд дорад."

#: tmp/addrepo.glade.h:11
msgid "Proxy pass_word"
msgstr "Пар_оли Proxy"

#: tmp/addrepo.glade.h:12
msgid "Proxy u_sername"
msgstr "Но_ми корбари Proxy"

#: tmp/addrepo.glade.h:13
msgid "Repository _URL"
msgstr "Феҳрасти _URL"

#: tmp/addrepo.glade.h:14
msgid "Select A Directory"
msgstr "Интихоби феҳраст"

#: tmp/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr "URL ҳамчун рӯйхати алоқаҳои иловагӣ"

#: tmp/addrepo.glade.h:16
msgid "_Directory"
msgstr "Феҳраст"

#: tmp/addrepo.glade.h:17
msgid "_Options"
msgstr "_Интихобҳо"

#: tmp/addrepo.glade.h:18
msgid "_Partition"
msgstr "_Қисми диск"

#: tmp/addrepo.glade.h:19
msgid "_Path"
msgstr "_Роҳча"

#: tmp/addrepo.glade.h:20
msgid "_Server"
msgstr "_Хидматгоҳ"

#: tmp/anaconda.glade.h:1
msgid "Reboo_t"
msgstr ""

#: tmp/anaconda.glade.h:3
msgid "_Next"
msgstr "_Навбатӣ"

#: tmp/autopart.glade.h:1
#, fuzzy
msgid "<b>Resize _target (in MB):</b>"
msgstr "<b> _Тағйир диҳед:</b>"

#: tmp/autopart.glade.h:3
msgid "Re_view and modify partitioning layout"
msgstr ""

#: tmp/autopart.glade.h:4
msgid "Volume to Resize"
msgstr ""

#: tmp/autopart.glade.h:5
msgid "What drive would you like to _boot this installation from?"
msgstr ""

#: tmp/autopart.glade.h:6
msgid ""
"Which partition would you like to resize to make room for your installation?"
msgstr ""

#: tmp/autopart.glade.h:7
msgid "_Advanced storage configuration"
msgstr "_Танзимоти захирагоҳи иловагӣ"

#: tmp/autopart.glade.h:8
msgid "_Encrypt system"
msgstr "_Рамзгузории система"

#: tmp/autopart.glade.h:10
msgid "_Select the drive(s) to use for this installation."
msgstr ""

#: tmp/blwhere.glade.h:1
msgid "/boot"
msgstr "/boot"

#: tmp/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr "Дараҷаи дастгоҳи BIOS"

#: tmp/blwhere.glade.h:3
msgid "Boot loader device"
msgstr "Дастгоҳи боркунандаи Boot"

#: tmp/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr "Дастгоҳи аввалини BIOS:"

#: tmp/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr "Дастгоҳи чоруми BIOS:"

#: tmp/blwhere.glade.h:6
msgid "MBR"
msgstr "MBR"

#: tmp/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr "Дастгоҳи дуюми BIOS:"

#: tmp/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr "Дастгоҳи сеюми BIOS:"

#: tmp/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr ""

#: tmp/detailed-dialog.glade.h:1
msgid "Info"
msgstr "Иттилооти"

#: tmp/detailed-dialog.glade.h:2
msgid "_Details"
msgstr "_Тафсилотҳо"

#: tmp/exnSave.glade.h:1
msgid "Bug _description"
msgstr "Тафсилоти _хато"

#: tmp/exnSave.glade.h:2
msgid "Destination _file"
msgstr "Номи _файл"

#: tmp/exnSave.glade.h:3
msgid ""
"Local storage device\n"
"Local disk\n"
"Remote server (scp)"
msgstr ""

#: tmp/exnSave.glade.h:6
msgid "Please choose a destination for saving your traceback."
msgstr ""

#: tmp/exnSave.glade.h:8
msgid "Select A File"
msgstr "Интихоби файл"

#: tmp/exnSave.glade.h:9
msgid "_Host (host:port)"
msgstr "_Шабака (host:port)"

#: tmp/exnSave.glade.h:10
msgid "_Password"
msgstr "_Парол"

#: tmp/exnSave.glade.h:11
msgid "_User name"
msgstr "_Номи корбар"

#: tmp/instkey.glade.h:2
#, no-c-format
msgid "%(instkey)s:"
msgstr "%(instkey)s:"

#: tmp/instkey.glade.h:4
#, no-c-format
msgid "Please enter your %(instkey)s."
msgstr "Лутфан, % худро ворид кунед."

#: tmp/iscsi-config.glade.h:1
msgid "<b>CHAP _Password:</b>"
msgstr "<b>Номи  _пароли CHAP:</b>"

#: tmp/iscsi-config.glade.h:2
msgid "<b>CHAP _Username:</b>"
msgstr "<b>Номи  _корбари CHAP:</b>"

#: tmp/iscsi-config.glade.h:3
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "<b>Баргаштани номи  _пароли CHAP:</b>"

#: tmp/iscsi-config.glade.h:4
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "<b>Баргаштани номи  _корбари CHAP:</b>"

#: tmp/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr "<b>_Нишонаи суроғаи IP:</b>"

#: tmp/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr "<b>Номи _iSCSI Initiator:</b>"

#: tmp/iscsi-config.glade.h:9
msgid "_Add target"
msgstr "_Иловаи нишона"

#: tmp/liveinst.desktop.in.h:1
msgid "Install"
msgstr "Сабт кунед"

#: tmp/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr "CD-диски зиндаро ба компютер сабт намоед"

#: tmp/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr "Ба компютер сабт намоед"

#: tmp/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""

#: tmp/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""

#: tmp/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr "Тасдиқи гузарвожа:"

#: tmp/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr ""

#: tmp/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr "Вориди гузарвожа:"

#: tmp/netconfig.glade.h:2
msgid "<b>Gateway:</b>"
msgstr "<b>Суроҳаи хидматгоҳ:</b>"

#: tmp/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr "<b>Суроғаи IPv4:</b>"

#: tmp/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr "<b>Суроғаи IPv6:</b>"

#: tmp/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr "<b>Номи хидматгоҳ:</b>"

#: tmp/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr "<b>_Интерфейс:</b>"

#: tmp/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr "Пуштибонии IPv_4 фаъол созед"

#: tmp/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr "Пуштибонии IPv_6 фаъол созед"

#: tmp/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr ""

#: tmp/network.glade.h:1
msgid "Hostname:"
msgstr "Номи шабака:"

#: tmp/network.glade.h:2
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""

#: tmp/tasksel.glade.h:1
msgid "Customize _later"
msgstr "_Баъдтар танзим кунед"

#: tmp/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: tmp/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""

#: tmp/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""

#: tmp/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr "_Иловаи феҳрастҳои нармафзори иловагӣ"

#: tmp/tasksel.glade.h:7
msgid "_Customize now"
msgstr "_Танзим кунед"

#: tmp/tasksel.glade.h:8
msgid "_Modify repository"
msgstr "_Тағйири феҳраст"

#: tmp/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr "<b>Рақами дастгоҳ:</b>"

#: tmp/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#: tmp/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: tmp/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "Иловаи дастгоҳи FCP"

#. generated from lang-table
msgid "Afrikaans"
msgstr "Забони африкоӣ"

#. generated from lang-table
msgid "Arabic"
msgstr "Забони арабӣ"

#. generated from lang-table
msgid "Assamese"
msgstr "Забони асамесӣ"

#. generated from lang-table
msgid "Bengali"
msgstr "Забони бенгалӣ"

#. generated from lang-table
msgid "Bengali(India)"
msgstr "Забони бенгалӣ (Ҳиндустон)"

#. generated from lang-table
msgid "Bulgarian"
msgstr "Забони булғорӣ"

#. generated from lang-table
msgid "Catalan"
msgstr "Забони каталанӣ"

#. generated from lang-table
msgid "Chinese(Simplified)"
msgstr "Забони хитоӣ (оддӣ)"

#. generated from lang-table
msgid "Chinese(Traditional)"
msgstr "Забони хитоӣ (анъанавӣ)"

#. generated from lang-table
msgid "Croatian"
msgstr "Забони хорватӣ"

#. generated from lang-table
msgid "Czech"
msgstr "Забони чехӣ"

#. generated from lang-table
msgid "Danish"
msgstr "Забони даннӣ"

#. generated from lang-table
msgid "Dutch"
msgstr "Забони даниягӣ"

#. generated from lang-table
msgid "English"
msgstr "Забони англисӣ"

#. generated from lang-table
msgid "Estonian"
msgstr "Забони эстонӣ"

#. generated from lang-table
msgid "Finnish"
msgstr "Забони финнӣ"

#. generated from lang-table
msgid "French"
msgstr "Забони франсавӣ"

#. generated from lang-table
msgid "German"
msgstr "Забони немисӣ"

#. generated from lang-table
msgid "Greek"
msgstr "Забони юнонӣ"

#. generated from lang-table
msgid "Gujarati"
msgstr "Забони гуҷаратӣ"

#. generated from lang-table
msgid "Hebrew"
msgstr "Яҳудӣ"

#. generated from lang-table
msgid "Hindi"
msgstr "Забони ҳиндӣ"

#. generated from lang-table
msgid "Hungarian"
msgstr "Забони венгерӣ"

#. generated from lang-table
msgid "Icelandic"
msgstr "Забони исландӣ"

#. generated from lang-table
msgid "Iloko"
msgstr "Забони илоко"

#. generated from lang-table
msgid "Indonesian"
msgstr "Забони индонезӣ"

#. generated from lang-table
msgid "Italian"
msgstr "Забони италиёӣ"

#. generated from lang-table
msgid "Japanese"
msgstr "Забони японӣ"

#. generated from lang-table
msgid "Kannada"
msgstr "Забони каннадӣ"

#. generated from lang-table
msgid "Korean"
msgstr "Забони кореягӣ"

#. generated from lang-table
msgid "Macedonian"
msgstr "Забони мақдунӣ"

#. generated from lang-table
#, fuzzy
msgid "Maithili"
msgstr "Забони маратӣ"

#. generated from lang-table
msgid "Malay"
msgstr "Забони малайӣ"

#. generated from lang-table
msgid "Malayalam"
msgstr "Забони малайяламӣ"

#. generated from lang-table
msgid "Marathi"
msgstr "Забони маратӣ"

#. generated from lang-table
#, fuzzy
msgid "Nepali"
msgstr "Забони бенгалӣ"

#. generated from lang-table
msgid "Norwegian(Bokmål)"
msgstr "Забони норвегиягӣ (Букмалӣ)"

#. generated from lang-table
msgid "Northern Sotho"
msgstr "Забони соттои шимолӣ"

#. generated from lang-table
msgid "Oriya"
msgstr "Забони ориёӣ"

#. generated from lang-table
msgid "Persian"
msgstr "Забони форсӣ"

#. generated from lang-table
msgid "Polish"
msgstr "Забони полякӣ"

#. generated from lang-table
msgid "Portuguese"
msgstr "Забони португалӣ"

#. generated from lang-table
msgid "Portuguese(Brazilian)"
msgstr "Забони португалӣ (Бразилиягӣ)"

#. generated from lang-table
msgid "Punjabi"
msgstr "Забони панҷабӣ"

#. generated from lang-table
msgid "Romanian"
msgstr "Забони руминӣ"

#. generated from lang-table
msgid "Russian"
msgstr "Забони русӣ"

#. generated from lang-table
msgid "Serbian"
msgstr "Забони сербӣ"

#. generated from lang-table
msgid "Serbian(Latin)"
msgstr "Забони сербӣ (лотинӣ)"

#. generated from lang-table
msgid "Sinhala"
msgstr "Забони шинхала"

#. generated from lang-table
msgid "Slovak"
msgstr "Забони словакӣ"

#. generated from lang-table
msgid "Slovenian"
msgstr "Забони словениягӣ"

#. generated from lang-table
msgid "Spanish"
msgstr "Забони испанӣ"

#. generated from lang-table
msgid "Swedish"
msgstr "Забони шведӣ"

#. generated from lang-table
#, fuzzy
msgid "Tajik"
msgstr "Забони тамилӣ"

#. generated from lang-table
msgid "Tamil"
msgstr "Забони тамилӣ"

#. generated from lang-table
msgid "Telugu"
msgstr "Забони телугу"

#. generated from lang-table
msgid "Turkish"
msgstr "Забони туркӣ"

#. generated from lang-table
msgid "Ukrainian"
msgstr "Забони украинӣ"

#. generated from lang-table
msgid "Vietnamese"
msgstr "Забони ветнамӣ"

#. generated from lang-table
msgid "Welsh"
msgstr "Забони вэлсӣ"

#. generated from lang-table
msgid "Zulu"
msgstr "Забони зулу"
